src/sssom/context.py

```diff
@@ -108,16 +108,8 @@ def get_default_metadata() -> Metadata:
             if "@id" in v and "@prefix" in v:
                 if v["@prefix"]:
                     prefix_map[key] = v["@id"]
-    for key in contxt_external["@context"]:
-        v = contxt_external["@context"][key]
-        if isinstance(v, str):
-            if key not in prefix_map:
-                prefix_map[key] = v
-            else:
-                if prefix_map[key] != v:
-                    logging.warning(
-                        f"{key} is already in prefix map ({prefix_map[key]}, but with a different value than {v}"
-                    )
+
+    prefix_map.update({(k, v) for k, v in contxt_external.items() if k not in prefix_map})
```

Comment:
This is the only line in this PR I can judge, the rest looks great.

In a second PR, we should consider dropping our expansion/contraction code and replacing it entirely with `curies` (basically, we would not need get_built_in_prefix_map() anymore) - we just create our `Converter` the way we want and then use the `contract` and `expand` methods.

For this here: can you tell me what part of the epm ends up in converter.prefix_map? Can you find a single example in epm with multiple prefixes and multiple uri_prefixes, and print the respective rows in the prefixmap?

Reply:
So basically, since there is so much stuff in epm (multiple URI expansions), we need to be much more careful in how we use the content of the epm. We cant just use the prefixmap, because we lose which prefix should be preferred when their are multiple "fitting URIs". I think you should basically import the converter independently to parsers.py, and use it in place of the prefixmap logic. In "get_built_in_prefix_map()", you just leave the contxt, and drop that "external" thing.

Reply:
> For this here: can you tell me what part of the epm ends up in converter.prefix_map? Can you find a single example in epm with multiple prefixes and multiple uri_prefixes, and print the respective rows in the prefixmap?

The `uri_prefix` is the default value for the `prefix_map`.  For e.g.:

`'3dmet': 'http://www.3dmet.dna.affrc.go.jp/cgi/show_data.php?acc='` is the prefix_map for the reference:

https://github.com/biopragmatics/bioregistry/blob/92d1de65aa439ce588a5b8199d58e06e5852eef5/exports/contexts/obo.epm.json#L1-L17

Reply:
> We cant just use the prefixmap, because we lose which prefix should be preferred when their are multiple "fitting URIs".

Yes, you are correct. The `"uri_prefix_synonyms"` get unnoticed in the process I propose in this PR. This will need a major overhaul of the code to be entirely dependent on `curies.Converter`

Reply:
So the prefix map, what exactly is in it when there are multiple prefixes or and multiple uri prefixes?

Reply:
```
{'3dmet': 'http://www.3dmet.dna.affrc.go.jp/cgi/show_data.php?acc=', '4dn.biosource': 'https://data.4dnucleome.org/biosources/', '4DN': 'https://data.4dnucleome.org/biosources/', '4dn.replicate': 'https://data.4dnucleome.org/experiment-set-replicates/', 'aaindex': 'http://www.genome.jp/dbget-bin/www_bget?aaindex:', 'AAO': 'http://purl.obolibrary.org/obo/AAO_', 'abcd': 'https://web.expasy.org/abcd/ABCD_', 'abm': 'https://www.abmgood.com/search?query=', 'abs': 'https://identifiers.org/abs:', 'ac': 'http://www.w3.org/ns/activitystreams#', 'aceview.worm': 'https://www.ncbi.nlm.nih.gov/IEB/Research/Acembly/av.cgi?db=worm&c=Gene&l=', 'adcad': 'https://purl.dataone.org/odo/ADCAD_', 'addexbio': 'https://www.addexbio.com/productdetail?pid=', 'addgene': 'http://addgene.org/', 'adms': 'http://www.w3.org/ns/adms#', 'ADO': 'http://purl.obolibrary.org/obo/ADO_', 'ADW': 'http://purl.obolibrary.org/obo/ADW_', 'AEO': 'http://purl.obolibrary.org/obo/AEO_', 'AEO_RETIRED': 'http://purl.obolibrary.org/obo/AEO_', 'aeon': 'http://purl.obolibrary.org/obo/AEON_', 'AERO': 'http://purl.obolibrary.org/obo/AERO_', 'affy.probeset': 'https://www.affymetrix.com/LinkServlet?probeset=', 'aftol.taxonomy': 'http://wasabi.lutzonilab.net/pub/displayTaxonInfo?aftol_id=', 'agricola': 'https://agricola.nal.usda.gov/vwebv/holdingsInfo?bibId=', 'AGR': 'https://agricola.nal.usda.gov/vwebv/holdingsInfo?bibId=', 'AGRICOLA_ID': 'https://agricola.nal.usda.gov/vwebv/holdingsInfo?bibId=', 'agrkb': 'https://www.alliancegenome.org/accession/', 'AGRO': 'http://purl.obolibrary.org/obo/AGRO_', 'agrovoc': 'http://aims.fao.org/aos/agrovoc/', 'agsd': 'http://www.genomesize.com/result_species.php?id=', 'aio': 'https://w3id.org/aio/', 'AISM': 'http://purl.obolibrary.org/obo/AISM_', 'alfred': 'https://alfred.med.yale.edu/alfred/recordinfo.asp?UNID=', 'allergome': 'http://www.allergome.org/script/dettaglio.php?id_molecule=', 'alzforum.mutation': 'https://www.alzforum.org/mutations/', 'Alzforum_mut': 'https://www.alzforum.org/mutations/', 'alzgene': 'http://www.alzgene.org/geneoverview.asp?geneid=', 'amoebadb': 'https://amoebadb.org/amoeba/app/record/gene/', 'AMPHX': 'http://purl.obolibrary.org/obo/AMPHX_', 'antibodyregistry': 'http://antibodyregistry.org/AB_', 'antweb': 'http://www.antweb.org/specimen.do?name=', 'anzctr': 'https://anzctr.org.au/Trial/Registration/TrialReview.aspx?ACTRN=', 'ACTRN': 'https://anzctr.org.au/Trial/Registration/TrialReview.aspx?ACTRN=', 'aop': 'https://aopwiki.org/aops/', 'aop.events': 'https://aopwiki.org/events/', 'aop.relationships': 'https://aopwiki.org/relationships/', 'aop.stressor': 'https://aopwiki.org/stressors/', 'apaonto': 'http://ontology.apa.org/apaonto/termsonlyOUT%20(5).owl#', 'apd': 'http://aps.unmc.edu/AP/database/query_output.php?ID=', 'aphidbase.transcript': 'http://bipaa.genouest.org/apps/grs-2.3/grs?reportID=aphidbase_transcript_report&objectID=', 'APO': 'http://purl.obolibrary.org/obo/APO_', 'APOLLO_SV': 'http://purl.obolibrary.org/obo/APOLLO_SV_', 'arachnoserver': 'http://www.arachnoserver.org/toxincard.html?id=', 'araport': 'https://www.arabidopsis.org/servlets/TairObject?type=locus&name=', 'archdb': 'http://sbi.imim.es/cgi-bin/archdb//loops.pl?loop=', 'ardb': 'http://ardb.cbcb.umd.edu/cgi/search.cgi?db=L&amp;field=ni&amp;term=', 'ark': 'http://n2t.net/ark:', 'ARO': 'http://purl.obolibrary.org/obo/ARO_', 'arrayexpress': 'https://www.ebi.ac.uk/arrayexpress/experiments/', 'arrayexpress.platform': 'https://www.ebi.ac.uk/arrayexpress/arrays/', 'arraymap': 'https://www.arraymap.org/pgx:', 'arxiv': 'https://arxiv.org/abs/', 'asap': 'http://asap.ahabs.wisc.edu/asap/feature_info.php?LocationID=WIS&FeatureID=', 'ascl': 'http://ascl.net/', 'asin': 'https://amzn.com/', 'aspgd.locus': 'http://www.aspergillusgenome.org/cgi-bin/locus.pl?dbid=', 'aspgd.protein': 'http://www.aspergillusgenome.org/cgi-bin/protein/proteinPage.pl?dbid=', 'asrp': 'http://asrp.cgrb.oregonstate.edu/db/sRNAdisplay.html?ASRP_id=', 'atc': 'http://www.whocc.no/atc_ddd_index/?code=', 'ATC_code': 'http://www.whocc.no/atc_ddd_index/?code=', 'ATTC': 'http://www.whocc.no/atc_ddd_index/?code=', 'atcc': 'https://www.atcc.org/products/', 'ATCC': 'https://www.atcc.org/products/', 'ATCC number': 'https://www.atcc.org/products/', 'ATCC(dna)': 'https://www.atcc.org/products/', 'ATCC(in host)': 'https://www.atcc.org/products/', 'atcvet': 'http://www.whocc.no/atcvet/atcvet_index/?code=', 'atfdb.family': 'http://www.bioguo.org/AnimalTFDB/family.php?fam=', 'ATO': 'http://purl.obolibrary.org/obo/ATO_', 'atol': 'http://opendata.inra.fr/ATOL/ATOL_', 'autdb': 'http://autism.mindspec.org/GeneDetail/', 'babelon': 'https://w3id.org/babelon/', 'bacdive': 'https://bacdive.dsmz.de/strain/', 'bacmap.biog': 'http://bacmap.wishartlab.com/organisms/', 'bacmap.map': 'https://identifiers.org/bacmap.map:', 'bactibase': 'http://bactibase.hammamilab.org/', 'bao': 'http://bioportal.bioontology.org/ontologies/BAO/bao:BAO_', 'BAO': 'http://bioportal.bioontology.org/ontologies/BAO/bao:BAO_', 'bartoc': 'https://bartoc.org/en/node/', 'bbkg': 'https://bbp.epfl.ch/nexus/web/studios/public/', 'bbtp': 'https://bbp.epfl.ch/nexus/web/studios/public/topological-sampling/studios/data:', 'BCGO': 'http://purl.obolibrary.org/obo/BCGO_', 'bcio': 'https://w3id.org/BCI-ontology#', 'BCO': 'http://purl.obolibrary.org/obo/BCO_', 'bcrc': 'https://catalog.bcrc.firdi.org.tw/BcrcContent?bid=', 'bcrj': 'http://bcrj.org.br/celula/', 'bdgp.insertion': 'http://flypush.imgen.bcm.tmc.edu/pscreen/details.php?line=', 'bdsc': 'https://bdsc.indiana.edu/stocks/', 'beetlebase': 'http://beetlebase.org/cgi-bin/gbrowse/BeetleBase3.gff3/?name=', 'begdb': 'http://www.begdb.com/index.php?action=oneMolecule&state=show&id=', 'BFO': 'http://purl.obolibrary.org/obo/BFO_', 'bgee.family': 'http://bgee.unil.ch/bgee/bgee?page=gene_family&action=family_details&gene_family_id=', 'bgee.organ': 'http://bgee.unil.ch/bgee/bgee?page=anatomy&action=organ_details&organ_children=on&organ_id=', 'bgee.stage': 'http://bgee.unil.ch/bgee/bgee?page=anatomy&action=organs&stage_children=on&stage_id=', 'bibo': 'http://purl.org/ontology/bibo/', 'BiDO': 'http://purl.org/spar/bido/', 'bigg.compartment': 'http://bigg.ucsd.edu/compartments/', 'bigg.metabolite': 'http://bigg.ucsd.edu/models/universal/metabolites/', 'bigg.model': 'http://bigg.ucsd.edu/models/', 'bigg.reaction': 'http://bigg.ucsd.edu/models/universal/reactions/', 'BILA': 'http://purl.obolibrary.org/obo/BILA_', 'bind': 'http://www.bind.ca/Action?identifier=bindid&idsearch=', 'bindingdb': 'http://www.bindingdb.org/compact/', 'biocarta.pathway': 'https://cgap.nci.nih.gov/Pathways/BioCarta/', 'biocatalogue.service': 'https://www.biocatalogue.org/services/', 'biocyc': 'http://biocyc.org/getid?id=', 'biofactoid': 'https://biofactoid.org/document/', 'biogrid': 'http://thebiogrid.org/', 'biogrid.interaction': 'https://thebiogrid.org/interaction/', 'biokc': 'https://biokb.lcsb.uni.lu/fact/', 'biolegend': 'https://www.biolegend.com/Default.aspx?ID=6664&productid=', 'biolink': 'https://w3id.org/biolink/vocab/', 'biomagresbank': 'http://www.bmrb.wisc.edu/data_library/generate_summary.php?bmrbId=', 'biominder': 'https://datalab.rwth-aachen.de/MINDER/resource/', 'biomodels.db': 'https://www.ebi.ac.uk/biomodels/', 'BIOMD': 'https://www.ebi.ac.uk/biomodels/', 'KISAO': 'http://purl.obolibrary.org/obo/KISAO_', 'biomodels.kisao': 'http://purl.obolibrary.org/obo/KISAO_', 'kisao': 'http://purl.obolibrary.org/obo/KISAO_', 'biomodels.teddy': 'https://www.ebi.ac.uk/ols/ontologies/teddy/terms?iri=http://identifiers.org/teddy/TEDDY_', 'teddy': 'https://www.ebi.ac.uk/ols/ontologies/teddy/terms?iri=http://identifiers.org/teddy/TEDDY_', 'biomodels.vocabulary': 'http://biomodels.net/rdf/vocabulary.rdf#', 'bionumbers': 'https://bionumbers.hms.harvard.edu/bionumber.aspx?id=', 'biopixie': 'http://avis.princeton.edu/pixie/viewgraph.php?graphID=', 'bioportal': 'http://bioportal.bioontology.org/ontologies/', 'bioregistry': 'https://bioregistry.io/registry/', 'bioregistry.collection': 'https://bioregistry.io/collection/', 'bioregistry.registry': 'https://bioregistry.io/metaregistry/', 'bioregistry.schema': 'https://bioregistry.io/schema/#', 'biorxiv': 'http://dx.doi.org/10.1101/', 'biosample': 'https://www.ebi.ac.uk/biosamples/sample/', 'biosamples': 'https://www.ebi.ac.uk/biosamples/sample/', 'bioschemas': 'https://bioschemas.org/profiles/', 'bioschema': 'https://bioschemas.org/profiles/', 'biosimulations': 'https://biosimulations.org/projects/', 'biosimulators': 'https://biosimulators.org/simulators/', 'biostudies': 'https://www.ebi.ac.uk/biostudies/studies/', 'biosystems': 'https://www.ncbi.nlm.nih.gov/biosystems/', 'biotools': 'https://bio.tools/', 'biozil': 'https://www.biozol.de/en/product/', 'birnlex': 'http://uri.neuinfo.org/nif/nifstd/birnlex_', 'BiRO': 'http://purl.org/spar/biro/', 'bitbucket': 'https://bitbucket.org/', 'bitterdb.cpd': 'http://bitterdb.agri.huji.ac.il/bitterdb/compound.php?id=', 'bitterdb.rec': 'http://bitterdb.agri.huji.ac.il/Receptor.php?id=', 'bko': 'http://www.sbgnbricks.org/BKO/full/entry/all/BKO:', 'bmrb': 'https://identifiers.org/bmrb:', 'bmrb.restraint': 'https://restraintsgrid.bmrb.io/NRG/MRGridServlet?block_text_type=2-parsed&db_username=wattos1&file_detail=2-parsed&format=n%2Fa&program=STAR&request_type=block&subtype=full&type=entry&mrblock_id=', 'bold.taxonomy': 'http://www.boldsystems.org/index.php/Taxbrowser_Taxonpage?taxid=', 'BOOTSTREP': 'http://purl.obolibrary.org/obo/BOOTSTREP_', 'brenda.ligand': 'https://www.brenda-enzymes.de/ligand.php?brenda_ligand_id=', 'brenda.ligandgroup': 'https://www.brenda-enzymes.de/ligand.php?brenda_group_id=', 'bridgedb': 'http://vocabularies.bridgedb.org/ops#', 'broad': 'https://www.broadinstitute.org/annotation/genome/magnaporthe_grisea/GeneDetails.html?sp=', 'bs': 'https://biopragmatics.github.io/providers/bs/', 'BSPO': 'http://purl.obolibrary.org/obo/BSPO_', 'BTO': 'http://purl.obolibrary.org/obo/BTO_', 'bugbase.expt': 'http://bugs.sgul.ac.uk/bugsbase/tabs/experiment.php?action=view&expt_id=', 'bugbase.protocol': 'http://bugs.sgul.ac.uk/bugsbase/tabs/protocol.php?action=view&protocol_id=', 'C4O': 'http://purl.org/spar/c4o/', 'cabri': 'https://identifiers.org/cabri:', 'cadsr': 'https://identifiers.org/cadsr:', 'caloha': 'https://www.nextprot.org/term/', 'cameo': 'https://www.cameo3d.org/sp/targets/target/', 'caps': 'http://www.bioinsilico.org/cgi-bin/CAPSDB/getCAPScluster?nidcl=', 'CARO': 'http://purl.obolibrary.org/obo/CARO_', 'cas': 'https://commonchemistry.cas.org/detail?cas_rn=', 'CASID': 'https://commonchemistry.cas.org/detail?cas_rn=', 'CAS_RN': 'https://commonchemistry.cas.org/detail?cas_rn=', 'SECONDARY_CAS_RN': 'https://commonchemistry.cas.org/detail?cas_rn=', 'cas_id': 'https://commonchemistry.cas.org/detail?cas_rn=', 'casspc': 'https://researcharchive.calacademy.org/research/ichthyology/catalog/fishcatget.asp?spid=', 'cath': 'http://www.cathdb.info/cathnode/', 'cath.superfamily': 'http://www.cathdb.info/cathnode/', 'cath.domain': 'http://www.cathdb.info/domain/', 'cazy': 'https://identifiers.org/cazy:', 'cba': 'https://europepmc.org/article/CBA/', 'cbioportal': 'https://www.cbioportal.org/study/summary?id=', 'ccdc': 'https://identifiers.org/ccdc:', 'ccds': 'http://www.ncbi.nlm.nih.gov/CCDS/CcdsBrowse.cgi?REQUEST=CCDS&DATA=', 'ccf': 'http://purl.org/ccf/', 'ccle': 'https://www.cbioportal.org/patient?studyId=ccle_broad_2019&caseId=', 'ccle.cell': 'https://www.cbioportal.org/patient?studyId=ccle_broad_2019&caseId=', 'cco': 'https://www.ebi.ac.uk/ols4/ontologies/cco/terms?obo_id=CCO:', 'ccrid': 'http://www.cellresource.cn/cellsearch.aspx?sc=1&where=', 'CDAO': 'http://purl.obolibrary.org/obo/CDAO_', 'cdd': 'https://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=', 'CDNO': 'http://purl.obolibrary.org/obo/CDNO_', 'cdpd': 'http://webprod3.hc-sc.gc.ca/dpd-bdpp/info.do?lang=eng&code=', 'cell_biolabs': 'https://www.cellbiolabs.com/search?keywords=', 'cell_model_passport': 'https://cellmodelpassports.sanger.ac.uk/passports/', 'cellimage': 'http://cellimagelibrary.org/images/', 'cellopub': 'https://web.expasy.org/cellosaurus/cellopub/', 'cellosaurus': 'https://www.cellosaurus.org/CVCL_', 'CVCL': 'https://www.cellosaurus.org/CVCL_', 'cellosaurus.resource': 'https://bioregistry.io/metaregistry/cellosaurus/', 'cellrepo': 'https://cellrepo.ico2s.org/repositories/', 'cemo': 'https://biopragmatics.github.io/providers/cemo/', 'CEPH': 'http://purl.obolibrary.org/obo/CEPH_', 'cgd': 'http://www.candidagenome.org/cgi-bin/locus.pl?dbid=', 'cgnc': 'http://birdgenenames.org/cgnc/GeneReport?id=', 'BirdBase': 'http://birdgenenames.org/cgnc/GeneReport?id=', 'CGNC': 'http://birdgenenames.org/cgnc/GeneReport?id=', 'cgsc': 'http://cgsc.biology.yale.edu/Site.php?ID=', 'charprot': 'http://www.jcvi.org/charprotdb/index.cgi/view/', 'CHEBI': 'http://purl.obolibrary.org/obo/CHEBI_', 'CHEBIID': 'http://purl.obolibrary.org/obo/CHEBI_', 'ChEBI': 'http://purl.obolibrary.org/obo/CHEBI_', 'chembank': 'http://chembank.broadinstitute.org/chemistry/viewMolecule.htm?cbid=', 'chembl': 'https://www.ebi.ac.uk/chembl/entity/', 'ChEMBL': 'https://www.ebi.ac.uk/chembl/entity/', 'ChEMBL-Targets': 'https://www.ebi.ac.uk/chembl/entity/', 'ChEMBL_ID': 'https://www.ebi.ac.uk/chembl/entity/', 'chembl.target': 'https://www.ebi.ac.uk/chembl/entity/', 'chembl.cell': 'https://www.ebi.ac.uk/chembl/cell_line_report_card/', 'ChEMBL-Cells': 'https://www.ebi.ac.uk/chembl/cell_line_report_card/', 'chembl.compound': 'http://rdf.ebi.ac.uk/resource/chembl/molecule/', 'chemdb': 'http://cdb.ics.uci.edu/cgibin/ChemicalDetailWeb.py?chemical_id=', 'chemidplus': 'https://chem.nlm.nih.gov/chemidplus/rn/', 'ChemIDplus': 'https://chem.nlm.nih.gov/chemidplus/rn/', 'CHEMINF': 'http://purl.obolibrary.org/obo/CHEMINF_', 'ChEMROF': 'https://chemkg.github.io/chemrof/', 'chemspider': 'http://rdf.chemspider.com/', 'ChemSpiderID': 'http://rdf.chemspider.com/', 'Chemspider': 'http://rdf.chemspider.com/', 'CHIRO': 'http://purl.obolibrary.org/obo/CHIRO_', 'CHMO': 'http://purl.obolibrary.org/obo/CHMO_', 'CIDO': 'http://purl.obolibrary.org/obo/CIDO_', 'CIO': 'http://purl.obolibrary.org/obo/CIO_', 'citexplore': 'https://europepmc.org/article/CTX/', 'CTX': 'https://europepmc.org/article/CTX/', 'CiTO': 'http://purl.org/spar/cito/', 'civic.aid': 'https://civicdb.org/links/assertions/', 'civic.did': 'https://civicdb.org/links/diseases/', 'civic.eid': 'https://civicdb.org/links/evidence/', 'civic.gid': 'https://civicdb.org/links/gene/', 'civic.mpid': 'https://civicdb.org/links/molecular_profile/', 'civic.sid': 'https://civicdb.org/links/sources/', 'civic.tid': 'https://civicdb.org/links/drugs/', 'civic.vgid': 'https://civicdb.org/links/variant_group/', 'civic.vid': 'https://civicdb.org/links/variant/', 'CL': 'http://purl.obolibrary.org/obo/CL_', 'CLAO': 'http://purl.obolibrary.org/obo/CLAO_', 'classyfire': 'http://classyfire.wishartlab.com/tax_nodes/C', 'CHEMONTID': 'http://classyfire.wishartlab.com/tax_nodes/C', 'clb': 'https://www.checklistbank.org/dataset/', 'cldb': 'https://identifiers.org/cldb:', 'hypercldb': 'https://identifiers.org/cldb:', 'clingene': 'https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=', 'CAID': 'https://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid=', 'clinicaltrials': 'https://www.clinicaltrials.gov/study/', 'NCT': 'https://www.clinicaltrials.gov/study/', 'clinicaltrial': 'https://www.clinicaltrials.gov/study/', 'ctgov': 'https://www.clinicaltrials.gov/study/', 'clinvar': 'https://www.ncbi.nlm.nih.gov/clinvar/variation/', 'clinvar.variant': 'https://www.ncbi.nlm.nih.gov/clinvar/variation/', 'clinvar.variation': 'https://www.ncbi.nlm.nih.gov/clinvar/variation/', 'clinvar.record': 'http://www.ncbi.nlm.nih.gov/clinvar/', 'clinvar.submission': 'http://www.ncbi.nlm.nih.gov/clinvar?term=', 'clinvar.submitter': 'https://www.ncbi.nlm.nih.gov/clinvar/submitters/', 'CLO': 'http://purl.obolibrary.org/obo/CLO_', 'cls': 'https://cls.shop/', 'CLYH': 'http://purl.obolibrary.org/obo/CLYH_', 'CMF': 'http://purl.obolibrary.org/obo/CMF_', 'CMO': 'http://purl.obolibrary.org/obo/CMO_', 'cmpo': 'http://www.ebi.ac.uk/cmpo/CMPO_', 'CO_320': 'https://cropontology.org/rdf/CO_320:', 'CO_321': 'https://cropontology.org/rdf/CO_321:', 'CO_322': 'https://cropontology.org/rdf/CO_322:', 'CO_323': 'https://cropontology.org/rdf/CO_323:', 'CO_324': 'https://cropontology.org/rdf/CO_324:', 'CO_325': 'https://cropontology.org/rdf/CO_325:', 'CO_326': 'https://cropontology.org/rdf/CO_326:', 'CO_327': 'https://cropontology.org/rdf/CO_327:', 'CO_330': 'https://cropontology.org/rdf/CO_330:', 'CO_331': 'https://cropontology.org/rdf/CO_331:', 'CO_333': 'https://cropontology.org/rdf/CO_333:', 'CO_334': 'https://cropontology.org/rdf/CO_334:', 'CO_335': 'https://cropontology.org/rdf/CO_335:', 'CO_336': 'https://cropontology.org/rdf/CO_336:', 'CO_337': 'https://cropontology.org/rdf/CO_337:', 'CO_338': 'https://cropontology.org/rdf/CO_338:', 'CO_339': 'https://cropontology.org/rdf/CO_339:', 'CO_340': 'https://cropontology.org/rdf/CO_340:', 'CO_341': 'https://cropontology.org/rdf/CO_341:', 'CO_343': 'https://cropontology.org/rdf/CO_343:', 'CO_345': 'https://cropontology.org/rdf/CO_345:', 'CO_346': 'https://cropontology.org/rdf/CO_346:', 'CO_347': 'https://cropontology.org/rdf/CO_347:', 'CO_348': 'https://cropontology.org/rdf/CO_348:', 'CO_350': 'https://cropontology.org/rdf/CO_350:', 'CO_356': 'https://cropontology.org/rdf/CO_356:', 'CO_357': 'https://cropontology.org/rdf/CO_357:', 'CO_358': 'https://cropontology.org/rdf/CO_358:', 'CO_359': 'https://cropontology.org/rdf/CO_359:', 'CO_360': 'https://cropontology.org/rdf/CO_360:', 'CO_365': 'https://cropontology.org/rdf/CO_365:', 'CO_366': 'https://cropontology.org/rdf/CO_366:', 'CO_367': 'https://cropontology.org/rdf/CO_367:', 'CO_370': 'https://cropontology.org/rdf/CO_370:', 'COB': 'http://purl.obolibrary.org/obo/COB_', 'coconut': 'https://coconut.naturalproducts.net/compound/coconut_id/', 'cog': 'https://www.ncbi.nlm.nih.gov/research/cog/cog/', 'COG_Cluster': 'https://www.ncbi.nlm.nih.gov/research/cog/cog/', 'cog.category': 'https://www.ncbi.nlm.nih.gov/research/cog/cogcategory/', 'cog.pathway': 'https://www.ncbi.nlm.nih.gov/research/cog/pathway/', 'col': 'https://www.checklistbank.org/dataset/3LR/taxon/', 'col.taiwan': 'https://taibnet.sinica.edu.tw/eng/taibnet_species_detail.php?name_code=', 'COLAO': 'http://purl.obolibrary.org/obo/COLAO_', 'colonatlas': 'http://www.coloncanceratlas.org/search_cell_line?cell_line=', 'combine.specifications': 'https://identifiers.org/combine.specifications:', 'come': 'https://www.flymine.org/come/entry?gn=', 'commoncoreontology': 'http://www.ontologyrepository.com/CommonCoreOntologies/', 'complexportal': 'https://www.ebi.ac.uk/complexportal/complex/', 'ComplexPortal': 'https://www.ebi.ac.uk/complexportal/complex/', 'comptox': 'https://comptox.epa.gov/dashboard/', 'conference': 'https://w3id.org/scholarlydata/ontology/conference-ontology.owl#', 'confident.event': 'https://www.confident-conference.org/index.php/Event:', 'confident.series': 'https://www.confident-conference.org/index.php/Event_Series:', 'conoserver': 'http://www.conoserver.org/?page=card&table=protein&id=', 'conso': 'https://pharmacome.github.io/conso/', 'cordis.article': 'https://cordis.europa.eu/article/id/', 'cordis.project': 'https://cordis.europa.eu/project/id/', 'coriell': 'http://ccr.coriell.org/Sections/Search/Sample_Detail.aspx?Ref=', 'corrdb': 'https://www.animalgenome.org/CorrDB/q/?id=CorrID:', 'corum': 'https://mips.helmholtz-muenchen.de/corum/?id=', 'cosmic': 'http://cancer.sanger.ac.uk/cosmic/gene/overview?ln=', 'cosmic.cell': 'https://cancer.sanger.ac.uk/cell_lines/sample/overview?id=', 'covid19': 'https://covid19.sfb.uit.no/api/records/', 'covoc': 'http://purl.obolibrary.org/obo/COVOC_', 'cpc': 'https://worldwide.espacenet.com/classification?locale=en_EP#!/CPC=', 'cpt': 'https://www.aapc.com/codes/cpt-codes/', 'ama-cpt': 'https://www.aapc.com/codes/cpt-codes/', 'crisprdb': 'http://crispr.i2bc.paris-saclay.fr/cgi-bin/crispr/SpecieProperties_db.cgi?Taxon_id[]=', 'CRO': 'http://purl.obolibrary.org/obo/CRO_', 'crop2ml': 'http://www.crop2ml.org/cropmdb/', 'cryoem': 'http://scipion.i2pc.es/ontology/CRYOEM_', 'cryptodb': 'https://cryptodb.org/cryptodb/app/record/gene/', 'csa': 'https://www.ebi.ac.uk/thornton-srv/databases/CSA/SearchResults.php?PDBID=', 'csd': 'https://identifiers.org/csd:', 'csp': 'http://purl.bioontology.org/ontology/CSP/', 'CRISP': 'http://purl.bioontology.org/ontology/CSP/', 'CRISP Thesaurus': 'http://purl.bioontology.org/ontology/CSP/', 'CRISP Thesaurus, 2006': 'http://purl.bioontology.org/ontology/CSP/', 'CSP2005': 'http://purl.bioontology.org/ontology/CSP/', 'cst': 'https://identifiers.org/cst:', 'cst.ab': 'https://identifiers.org/cst.ab:', 'cstr': 'https://cstr.cn/', 'CTENO': 'http://purl.obolibrary.org/obo/CTENO_', 'ctis': 'https://euclinicaltrials.eu/app/#/view/', 'CTO': 'http://purl.obolibrary.org/obo/CTO_', 'cubedb': 'https://identifiers.org/cubedb:', 'CVDO': 'http://purl.obolibrary.org/obo/CVDO_', 'd1id': 'https://identifiers.org/d1id:', 'dailymed': 'https://dailymed.nlm.nih.gov/dailymed/drugInfo.cfm?setid=', 'dandi': 'https://dandiarchive.org/dandiset/', 'darc': 'http://darcsite.genzentrum.lmu.de/darc/view.php?id=', 'dashr': 'http://lisanwanglab.org/DASHR/entry/', 'dashr.expression': 'https://identifiers.org/dashr.expression:', 'DataCite': 'http://purl.org/spar/datacite/', 'datacommons': 'https://datacommons.org/browser/', 'datanator.gene': 'https://www.datanator.info/gene/', 'datanator.metabolite': 'https://www.datanator.info/metabolite/', 'datanator.reaction': 'https://datanator.info/reaction/', 'datf': 'http://planttfdb.cbi.pku.edu.cn/tf.php?sp=Ath&did=', 'dbd': 'https://identifiers.org/dbd:', 'dbest': 'https://www.ncbi.nlm.nih.gov/nucest/', 'bdgp.est': 'https://www.ncbi.nlm.nih.gov/nucest/', 'dbg2introns': 'http://webapps2.ucalgary.ca/~groupii/cgi-bin/intron.cgi?name=', 'dbgap': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=', 'dbo': 'https://dbpedia.org/ontology/', 'dbprobe': 'https://www.ncbi.nlm.nih.gov/probe/?term=', 'dbsnp': 'https://www.ncbi.nlm.nih.gov/snp/', 'snp2tfbs': 'https://www.ncbi.nlm.nih.gov/snp/', 'dbvar.study': 'https://www.ncbi.nlm.nih.gov/dbvar/studies/', 'dbvar.studies': 'https://www.ncbi.nlm.nih.gov/dbvar/studies/', 'dbvar.variant': 'https://www.ncbi.nlm.nih.gov/dbvar/variants/', 'dbvar.variants': 'https://www.ncbi.nlm.nih.gov/dbvar/variants/', 'dc': 'http://purl.org/dc/elements/1.1/', 'dc.elements': 'http://purl.org/dc/elements/1.1/', 'dc11': 'http://purl.org/dc/elements/1.1/', 'dce': 'http://purl.org/dc/elements/1.1/', 'DC_CL': 'http://purl.obolibrary.org/obo/DC_CL_', 'dcat': 'http://www.w3.org/ns/dcat#', 'dcterms': 'http://purl.org/dc/terms/', 'dc.terms': 'http://purl.org/dc/terms/', 'dct': 'http://purl.org/dc/terms/', 'dctypes': 'http://purl.org/dc/dcmitype/', 'DDANAT': 'http://purl.obolibrary.org/obo/DDANAT_', 'ddinter.drug': 'http://ddinter.scbdd.com/ddinter/drug-detail/', 'ddinter.interaction': 'http://ddinter.scbdd.com/ddinter/interact/', 'DDPHENO': 'http://purl.obolibrary.org/obo/DDPHENO_', 'DeBiO': 'https://biopragmatics.github.io/debio/', 'decipher': 'https://www.deciphergenomics.org/syndrome/', 'degradome': 'http://degradome.uniovi.es/cgi-bin/protease/', 'DEO': 'http://purl.org/spar/deo/', 'depmap': 'https://depmap.org/portal/cell_line/', 'dermo': 'http://purl.obolibrary.org/obo/DERMO_', 'dev.ga4ghdos': 'https://dos-gdc.ucsc-cgp-dev.org/ga4gh/dos/v1/dataobjects/', 'dg.4503': 'https://gen3.biodatacatalyst.nhlbi.nih.gov/ga4gh/drs/v1/objects/', 'dg.4dfc': 'https://nci-crdc.datacommons.io/ga4gh/drs/v1/objects/', 'dg.6vts': 'https://jcoin.datacommons.io/ga4gh/drs/v1/objects/', 'dg.anv0': 'https://gen3.theanvil.io/ga4gh/drs/v1/objects/', 'dg.f82a1a': 'https://data.kidsfirstdrc.org/ga4gh/drs/v1/objects/', 'dg5b0d': 'https://data.bloodpac.org/ga4gh/drs/v1/objects/', 'dgrc': 'https://dgrc.bio.indiana.edu/product/View?product=', 'DHBA': 'https://biopragmatics.github.io/providers/dhba/', 'dicom': 'http://dicom.nema.org/resources/ontology/DCM/', 'dictybase': 'http://dictybase.org/gene/', 'dictyBase': 'http://dictybase.org/gene/', 'dictybase.gene': 'http://dictybase.org/gene/', 'dictybase.est': 'http://dictybase.org/db/cgi-bin/feature_page.pl?primary_id=', 'did': 'https://uniresolver.io/#did:', 'DIDEO': 'http://purl.obolibrary.org/obo/DIDEO_', 'DINTO': 'http://purl.obolibrary.org/obo/DINTO_', 'dip': 'https://dip.doe-mbi.ucla.edu/dip/DIPview.cgi?ID=', 'DISDRIV': 'http://purl.obolibrary.org/obo/DISDRIV_', 'diseaseclass': 'https://biopragmatics.github.io/providers/diseaseclass/', 'disprot': 'https://disprot.org/', 'disprot.region': 'https://disprot.org/', 'dlxb': 'https://doulix.com/biomodules/', 'dlxc': 'https://doulix.com/constructs/', 'DMBA': 'https://biopragmatics.github.io/providers/dmba/', 'doap': 'http://usefulinc.com/ns/doap#', 'DoCO': 'http://purl.org/spar/doco/', 'doi': 'http://dx.doi.org/', 'DOID': 'http://purl.obolibrary.org/obo/DOID_', 'do': 'http://purl.obolibrary.org/obo/DOID_', 'dommino': 'http://orion.rnet.missouri.edu/~nz953/DOMMINO/index.php/result/show_network/', 'door': 'http://csbl.bmb.uga.edu/DOOR/operon.php?id=', 'doqcs.model': 'http://doqcs.ncbs.res.in/template.php?&y=accessiondetails&an=', 'doqcs.pathway': 'http://doqcs.ncbs.res.in/template.php?&y=pathwaydetails&pn=', 'FBcv': 'http://purl.obolibrary.org/obo/FBcv_', 'dpv': 'http://www.dpvweb.net/dpv/showdpv.php?dpvno=', 'dragondb.allele': 'https://identifiers.org/dragondb.allele:', 'dragondb.dna': 'https://identifiers.org/dragondb.dna:', 'dragondb.locus': 'https://identifiers.org/dragondb.locus:', 'dragondb.protein': 'https://identifiers.org/dragondb.protein:', 'drduke': 'https://phytochem.nal.usda.gov/phytochem/chemicals/show/', 'drks': 'https://drks.de/search/en/trial/', 'DRON': 'http://purl.obolibrary.org/obo/DRON_', 'drsanv0': 'https://data.terra.bio/ga4gh/drs/v1/objects/', 'drsc': 'http://www.flyrnai.org/cgi-bin/RNAi_gene_lookup_public.pl?gname=', 'drugbank': 'http://www.drugbank.ca/drugs/', 'DRUGBANK_ID': 'http://www.drugbank.ca/drugs/', 'DrugBank': 'http://www.drugbank.ca/drugs/', 'drugbank.category': 'https://www.drugbank.ca/categories/', 'drugbank.salt': 'https://go.drugbank.com/salts/', 'drugbank.target': 'https://go.drugbank.com/salts/', 'drugbankv4.target': 'http://www.drugbank.ca/biodb/bio_entities/', 'drugcentral': 'http://drugcentral.org/drugcard/', 'Drug_Central': 'http://drugcentral.org/drugcard/', 'dsmz': 'https://www.dsmz.de/collection/catalogue/details/culture/', 'DSMZCellDive': 'https://www.dsmz.de/collection/catalogue/details/culture/', 'dso': 'https://www.datascienceontology.org/concept/', 'dto': 'http://www.drugtargetontology.org/dto/DTO_', 'DUO': 'http://purl.obolibrary.org/obo/DUO_', 'eaglei': 'http://hawaii.eagle-i.net/i/', 'easychair.cfp': 'https://easychair.org/cfp/', 'easychair.topic': 'https://easychair.org/cfp/topic.cgi?tid=', 'ebisc': 'https://cells.ebisc.org/', 'ecacc': 'https://www.phe-culturecollections.org.uk/products/celllines/generalcell/detail.jsp?collection=ecacc_gc&refId=', 'ECAO': 'http://purl.obolibrary.org/obo/ECAO_', 'eccode': 'https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=', 'EC': 'https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=', 'EC number': 'https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=', 'EC-CODE': 'https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=', 'ECCODE': 'https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=', 'EC_CODE': 'https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=', 'brenda': 'https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=', 'ec-code': 'https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=', 'intenz': 'https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=', 'kegg.enzyme': 'https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=', 'sabiork.ec': 'https://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec=', 'ecg': 'http://www.cvrgrid.org/files/ECGOntologyv1.owl#ECG_', 'ECGOntology': 'http://www.cvrgrid.org/files/ECGOntologyv1.owl#ECG_', 'echinobase': 'https://www.echinobase.org/entry/gene/showgene.do?method=display&geneId=', 'echobase': 'http://www.york.ac.uk/res/thomas/Gene.cfm?recordID=', 'ecmdb': 'http://ecmdb.ca/compounds/', 'ECMDB': 'http://ecmdb.ca/compounds/', 'ECO': 'http://purl.obolibrary.org/obo/ECO_', 'ECOCORE': 'http://purl.obolibrary.org/obo/ECOCORE_', 'ecocyc': 'https://ecocyc.org/gene?id=', 'ecogene': 'http://www.ecogene.org/gene/', 'ecoliwiki': 'https://identifiers.org/ecoliwiki:', 'ecso': 'http://purl.dataone.org/odo/ECSO_', 'ECTO': 'http://purl.obolibrary.org/obo/ECTO_', 'ecyano.entity': 'https://www.e-cyanobacterium.org/bcs/entity/', 'ecyano.experiment': 'https://www.e-cyanobacterium.org/experiments-repository/?action=detail&id=', 'ecyano.model': 'https://e-cyanobacterium.org/models/model/', 'ecyano.rule': 'https://e-cyanobacterium.org/bcs/rule/', 'edam': 'https://www.ebi.ac.uk/ols/ontologies/edam/terms?iri=http://edamontology.org/', 'edam.data': 'http://edamontology.org/data_', 'edam.format': 'http://edamontology.org/format_', 'edam.operation': 'http://edamontology.org/operation_', 'edam.topic': 'http://edamontology.org/topic_', 'edda': 'http://ontologies.dbmi.pitt.edu/edda/StudyDesigns.owl#', 'EFO': 'http://www.ebi.ac.uk/efo/EFO_', 'ega.dataset': 'https://www.ebi.ac.uk/ega/datasets/', 'ega.study': 'https://www.ebi.ac.uk/ega/studies/', 'eggnog': 'http://eggnog.embl.de/version_3.0/cgi/search.py?search_term_0=', 'EHDA': 'http://purl.obolibrary.org/obo/EHDA_', 'EHDAA': 'http://purl.obolibrary.org/obo/EHDAA_', 'EHDAA2': 'http://purl.obolibrary.org/obo/EHDAA2_', 'EHDAA2_RETIRED': 'http://purl.obolibrary.org/obo/EHDAA2_', 'HDAA2': 'http://purl.obolibrary.org/obo/EHDAA2_', 'RETIRED_EHDAA2': 'http://purl.obolibrary.org/obo/EHDAA2_', 'elm': 'https://identifiers.org/elm:', 'EMAP': 'http://purl.obolibrary.org/obo/EMAP_', 'EMAPA': 'http://purl.obolibrary.org/obo/EMAPA_', 'EMAPA_RETIRED': 'http://purl.obolibrary.org/obo/EMAPA_', 'EMAPS': 'https://www.informatics.jax.org/vocab/gxd/anatomy/EMAPS:', 'emdb': 'https://www.ebi.ac.uk/pdbe/entry/emdb/', 'emmo': 'http://emmo.info/emmo#EMMO_', 'emmo.cif': 'http://emmo.info/emmo/cif-core#', 'emolecules': 'https://reaxys.emolecules.com/cgi-bin/more?vid=', 'empiar': 'https://www.ebi.ac.uk/pdbe/emdb/empiar/entry/', 'emsl.project': 'https://www.emsl.pnnl.gov/project/', 'ena.embl': 'https://www.ebi.ac.uk/ena/browser/view/', 'bioproject': 'https://www.ebi.ac.uk/ena/browser/view/', 'ena': 'https://www.ebi.ac.uk/ena/browser/view/', 'insdc.run': 'https://www.ebi.ac.uk/ena/browser/view/', 'encode': 'https://www.encodeproject.org/', 'enm': 'http://purl.enanomapper.org/onto/ENM_', 'ENSEMBL': 'http://rdf.ebi.ac.uk/resource/ensembl/', 'Ensembl': 'http://rdf.ebi.ac.uk/resource/ensembl/', 'astd': 'http://rdf.ebi.ac.uk/resource/ensembl/', 'ensembl.bacteria': 'http://rdf.ebi.ac.uk/resource/ensembl/', 'ensembl.fungi': 'http://rdf.ebi.ac.uk/resource/ensembl/', 'ensembl.metazoa': 'https://metazoa.ensembl.org/id/', 'ensembl.plant': 'https://plants.ensembl.org/id/', 'ensembl.protist': 'https://protists.ensembl.org/id/', 'ensemblglossary': 'http://ensembl.org/glossary/ENSGLOSSARY_', 'envipath': 'https://envipath.org/package/', 'ENVO': 'http://purl.obolibrary.org/obo/ENVO_', 'enzo': 'https://www.enzolifesciences.com/', 'Enzo Life Sciences': 'https://www.enzolifesciences.com/', 'EO': 'http://purl.obolibrary.org/obo/EO_', 'eol': 'http://purl.org/obo/owlEOL_', 'eolife': 'https://eol.org/pages/', 'epd': 'https://epd.expasy.org/cgi-bin/epd/query_result.pl?out_format=NICE&Entry_0=', 'EPIO': 'http://purl.obolibrary.org/obo/EPIO_', 'EPO': 'http://purl.obolibrary.org/obo/EPO_', 'epso': 'http://www.case.edu/EpilepsyOntology.owl#', 'erm': 'https://nanocommons.github.io/identifiers/registry#', 'ERO': 'http://purl.obolibrary.org/obo/ERO_', 'eropmoscow': 'http://erop.inbi.ras.ru/result2.php?PepName=', 'erv': 'https://herv.img.cas.cz/s/', 'esldb': 'http://gpcr.biocomp.unibo.it/cgi-bin/predictors/esldb/dettagli.cgi?codice=', 'estdab': 'https://www.ebi.ac.uk/cgi-bin/ipd/estdab/print_cell.cgi?ESTDAB-', 'eu89h': 'http://data.europa.eu/89h/', 'euclinicaltrials': 'https://www.clinicaltrialsregister.eu/ctr-search/search?query=', 'EUCTR': 'https://www.clinicaltrialsregister.eu/ctr-search/search?query=', 'eugenes': 'http://eugenes.org:7072/.bin/fbidq.html?', 'EUPATH': 'http://purl.obolibrary.org/obo/EUPATH_', 'EV': 'http://purl.obolibrary.org/obo/EV_', 'exac.gene': 'http://exac.broadinstitute.org/gene/', 'exac.transcript': 'http://exac.broadinstitute.org/transcript/', 'exac.variant': 'http://exac.broadinstitute.org/variant/', 'ExO': 'http://purl.obolibrary.org/obo/ExO_', 'fabio': 'https://sparontologies.github.io/fabio/current/fabio.html#', 'facebase': 'https://www.facebase.org/data/record/#1/isa:dataset/accession=', 'fairsharing': 'https://fairsharing.org/', 'fairsharing.organization': 'https://fairsharing.org/organisations/', 'fairsharing.user': 'https://fairsharing.org/users/', 'faldo': 'http://biohackathon.org/resource/faldo#', 'FAO': 'http://purl.obolibrary.org/obo/FAO_', 'fao.asfis': 'https://www.fao.org/fishery/en/species/', 'asfis': 'https://www.fao.org/fishery/en/species/', 'FBbi': 'http://purl.obolibrary.org/obo/FBbi_', 'FBbt': 'http://purl.obolibrary.org/obo/FBbt_', 'FBbt_root': 'http://purl.obolibrary.org/obo/FBbt_', 'FBdv': 'http://purl.obolibrary.org/obo/FBdv_', 'fbol': 'https://identifiers.org/fbol:', 'fbrf': 'https://flybase.org/reports/FBrf', 'FBSP': 'http://purl.obolibrary.org/obo/FBSP_', 'fbtc': 'https://flybase.org/reports/FBtc', 'FlyBase_Cell_line': 'https://flybase.org/reports/FBtc', 'fcb': 'https://w3id.org/faircookbook/', 'fcsfree': 'https://fcs-free.org/fcs-database?', 'fhir.implementation': 'https://hl7.org/fhir/us/', 'FIDEO': 'http://purl.obolibrary.org/obo/FIDEO_', 'fishbase.species': 'https://www.fishbase.ca/summary/', 'fishbase': 'https://www.fishbase.ca/summary/', 'FIX': 'http://purl.obolibrary.org/obo/FIX_', 'FLOPO': 'http://purl.obolibrary.org/obo/FLOPO_', 'flowrepository': 'https://flowrepository.org/id/', 'FLU': 'http://purl.obolibrary.org/obo/FLU_', 'FlyBase': 'https://flybase.org/reports/', 'FB': 'https://flybase.org/reports/', 'fbcv': 'https://flybase.org/reports/', 'flymine.chromosome': 'https://www.flymine.org/flymine/report/ChromosomeBand/', 'FMA': 'http://purl.obolibrary.org/obo/FMA_', 'FMAID': 'http://purl.obolibrary.org/obo/FMA_', 'FMA_RETIRED': 'http://purl.obolibrary.org/obo/FMA_', 'foaf': 'http://xmlns.com/foaf/0.1/', 'FOBI': 'http://purl.obolibrary.org/obo/FOBI_', 'foodb.compound': 'http://foodb.ca/compounds/', 'foodb': 'http://foodb.ca/compounds/', 'foodb.food': 'https://foodb.ca/foods/', 'foodex2': 'http://data.food.gov.uk/codes/foodtype/id/', 'FOODON': 'http://purl.obolibrary.org/obo/FOODON_', 'fossilworks.journal': 'http://fossilworks.org/?a=referenceInfo&reference_no=', 'fossilworks.taxon': 'http://www.fossilworks.org/cgi-bin/bridge.pl?a=taxonInfo&taxon_no=', 'FOVT': 'http://purl.obolibrary.org/obo/FOVT_', 'fplx': 'https://sorgerlab.github.io/famplex/', 'famplex': 'https://sorgerlab.github.io/famplex/', 'FRAPO': 'http://purl.org/cerif/frapo/', 'FRBR': 'http://purl.org/vocab/frbr/core#', 'frbrer': 'http://iflastandards.info/ns/fr/frbr/frbrer/', 'fsnp': 'http://compbio.cs.queensu.ca/cgi-bin/compbio/search/main.cgi?search_mode=id&amp;id_type=snp_id&amp;id_val=', 'ftt': 'https://apps.usgs.gov/thesaurus/term-simple.php?thcode=3&code=', 'funcbase.fly': 'http://func.mshri.on.ca/fly/genes/list_functional_scores/', 'funcbase.human': 'http://func.mshri.on.ca/human/genes/list_functional_scores/', 'funcbase.mouse': 'http://func.mshri.on.ca/mouse/genes/list_functional_scores/', 'funcbase.yeast': 'http://func.mshri.on.ca/yeast/genes/list_functional_scores/', 'funderregistry': 'http://data.crossref.org/fundingdata/funder/10.13039/', 'FundRef': 'http://data.crossref.org/fundingdata/funder/10.13039/', 'crossref.funder': 'http://data.crossref.org/fundingdata/funder/10.13039/', 'fungidb': 'https://fungidb.org/fungidb/app/record/gene/', 'fungorum': 'http://www.indexfungorum.org/names/NamesRecord.asp?RecordID=', 'FYPO': 'http://purl.obolibrary.org/obo/FYPO_', 'ga4ghdos': 'https://dataguids.org/ga4gh/dos/v1/dataobjects/', 'gabi': 'https://identifiers.org/gabi:', 'gainesville.core': 'http://purl.org/gc/', 'galen': 'http://www.co-ode.org/ontologies/galen#', 'gateway': 'https://web.www.healthdatagateway.org/dataset/', 'GAZ': 'http://purl.obolibrary.org/obo/GAZ_', 'gbif': 'https://www.gbif.org/species/', 'gc': 'https://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/index.cgi?chapter=cgencodes#SG', 'gc_id': 'https://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/index.cgi?chapter=cgencodes#SG', 'gcst': 'https://www.ebi.ac.uk/gwas/studies/', 'gdc': 'https://portal.gdc.cancer.gov/cases/', 'gdsc': 'https://www.cancerrxgene.org/translation/Drug/', 'gear': 'https://umgear.org/p?id=', 'GECKO': 'http://purl.obolibrary.org/obo/GECKO_', 'gemet': 'https://www.eionet.europa.eu/gemet/en/concept/', 'genatlas': 'http://genatlas.medecine.univ-paris5.fr/fiche.php?symbol=', 'genbank': 'https://www.ncbi.nlm.nih.gov/nucleotide/', 'NCBI_gi': 'https://www.ncbi.nlm.nih.gov/nucleotide/', 'ncbi.gi': 'https://www.ncbi.nlm.nih.gov/nucleotide/', 'ncbigi': 'https://www.ncbi.nlm.nih.gov/nucleotide/', 'gendis': 'http://caps.ncbs.res.in/cgi-bin/mini/databases/gendis/sf.cgi?code=', 'genecards': 'https://www.genecards.org/cgi-bin/carddisp.pl?gene=', 'genecards.geneloc': 'http://genecards.weizmann.ac.il/geneloc-bin/marker_cards.pl?id=', 'genedb': 'https://www.genedb.org/gene/', 'genefarm': 'https://urgi.versailles.inra.fr/Genefarm/Gene/display_gene.htpl?GENE_ID=', 'GENEPIO': 'http://purl.obolibrary.org/obo/GENEPIO_', 'genetree': 'http://www.ensembl.org/Multi/GeneTree/Image?db=core;gt=', 'GENO': 'http://purl.obolibrary.org/obo/GENO_', 'genpept': 'https://identifiers.org/genpept:', 'genprop': 'https://www.ebi.ac.uk/interpro/genomeproperties/#', 'geo': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=', 'GEO': 'http://purl.obolibrary.org/obo/GEO_', 'geonames': 'https://www.geonames.org/', 'Geomames': 'https://www.geonames.org/', 'Geonamaes': 'https://www.geonames.org/', 'gfo': 'http://www.onto-med.de/ontologies/gfo.owl#', 'ghr': 'https://medlineplus.gov/genetics/condition/', 'giardiadb': 'https://giardiadb.org/giardiadb/app/record/gene/', 'github': 'https://github.com/', 'github.issue': 'https://bioregistry.io/resolve/github/issue/', 'github.pull': 'https://bioregistry.io/resolve/github/pull/', 'gitlab': 'https://gitlab.com/', 'glida.gpcr': 'http://pharminfo.pharm.kyoto-u.ac.jp/services/glida/gpcr_information.php?id=', 'glida.ligand': 'http://pharminfo.pharm.kyoto-u.ac.jp/services/glida/ligand_information.php?id=', 'glycoepitope': 'https://www.glycoepitope.jp/epitopes/', 'glycomapsdb': 'http://www.glycosciences.de/modeling/glycomapsdb/showdetails.php?mapid=', 'glyconavi': 'https://glyconavi.org/hub/?id=', 'glycopost': 'https://glycopost.glycosmos.org/entry/', 'glycosciencesdb': 'http://www.glycosciences.de/database/start.php?action=explore_linucsid&linucsid=', 'glycosciences.db': 'http://www.glycosciences.de/database/start.php?action=explore_linucsid&linucsid=', 'glygen': 'https://glygen.org/glycan/', 'glytoucan': 'https://glytoucan.org/Structures/Glycans/', 'glycomedb': 'https://glytoucan.org/Structures/Glycans/', 'gmd': 'https://identifiers.org/gmd:', 'gmd.analyte': 'http://gmd.mpimp-golm.mpg.de/Analytes/', 'gmd.gcms': 'http://gmd.mpimp-golm.mpg.de/Spectrums/', 'gmd.profile': 'http://gmd.mpimp-golm.mpg.de/profile/default.aspx?XemlId=', 'gmd.ref': 'http://gmd.mpimp-golm.mpg.de/ReferenceSubstances/', 'gnd': 'https://lobid.org/gnd/', 'GNO': 'http://purl.obolibrary.org/obo/GNO_', 'gnome': 'http://purl.obolibrary.org/obo/GNO_', 'gnomad': 'https://gnomad.broadinstitute.org/variant/', 'gnpis': 'https://urgi.versailles.inra.fr/gnpis/#result/term=', 'gnps.task': 'https://gnps.ucsd.edu/ProteoSAFe/status.jsp?task=', 'GO': 'http://purl.obolibrary.org/obo/GO_', 'gobp': 'http://purl.obolibrary.org/obo/GO_', 'gobpid': 'http://purl.obolibrary.org/obo/GO_', 'gocc': 'http://purl.obolibrary.org/obo/GO_', 'goccid': 'http://purl.obolibrary.org/obo/GO_', 'gomf': 'http://purl.obolibrary.org/obo/GO_', 'gomfid': 'http://purl.obolibrary.org/obo/GO_', 'go.gpi': 'http://geneontology.org/docs/gene-product-information-gpi-format/#', 'go.model': 'http://noctua.geneontology.org/editor/graph/gomodel:', 'go.ref': 'https://github.com/geneontology/go-site/blob/master/metadata/gorefs/README.md#goref', 'go.resource': 'https://bioregistry.io/metaregistry/go/', 'GOCHE': 'https://biopragmatics.github.io/providers/goche/', 'go.chebi': 'https://biopragmatics.github.io/providers/goche/', 'go.chemical': 'https://biopragmatics.github.io/providers/goche/', 'go.chemicals': 'https://biopragmatics.github.io/providers/goche/', 'gold': 'https://gold.jgi.doe.gov/resolver?id=', 'gold.meta': 'https://gold.jgi.doe.gov/resolver?id=', 'gold.genome': 'http://www.genomesonline.org/cgi-bin/GOLD/GOLDCards.cgi?goldstamp=', 'goldbook': 'http://dx.doi.org/10.1351/goldbook.', 'google.book': 'https://books.google.com/books?id=', 'google.patent': 'https://www.google.com/patents/', 'patent': 'https://www.google.com/patents/', 'gorel': 'http://purl.obolibrary.org/obo/GOREL_', 'gpcrdb': 'https://gpcrdb.org/protein/', 'gpcrnava': 'http://nava.liacs.nl/cgi-bin/nava.py?id=', 'gpmdb': 'http://gpmdb.thegpm.org/~/dblist_gpmnum/gpmnum=', 'graingenes.reference': 'https://wheat.pw.usda.gov/cgi-bin/GG3/report.cgi?class=reference&name=', 'graingenes.symbol': 'http://wheat.pw.usda.gov/report?class=gene;name=', 'gramene.gene': 'http://www.gramene.org/db/genes/search_gene?acc=', 'GR_GENE': 'http://www.gramene.org/db/genes/search_gene?acc=', 'GRO': 'http://purl.obolibrary.org/obo/GRO_', 'cpga': 'http://purl.obolibrary.org/obo/GRO_', 'gro-cpga': 'http://purl.obolibrary.org/obo/GRO_', 'gramene.protein': 'http://www.gramene.org/db/protein/protein_search?protein_id=', 'GR_PROTEIN': 'http://www.gramene.org/db/protein/protein_search?protein_id=', 'gramene.qtl': 'http://www.gramene.org/db/qtl/qtl_display?qtl_accession_id=', 'GR_QTL': 'http://www.gramene.org/db/qtl/qtl_display?qtl_accession_id=', 'gramene.reference': 'http://www.gramene.org/db/literature/pub_search?ref_id=', 'gramene.ref': 'http://www.gramene.org/db/literature/pub_search?ref_id=', 'gramene.taxonomy': 'https://archive.gramene.org/db/ontology/search?id=GR_tax:', 'grassbase': 'https://www.kew.org/data/grasses-db/www/', 'greengenes': 'http://greengenes.lbl.gov/cgi-bin/show_one_record_v2.pl?prokMSA_id=', 'grid': 'https://www.grid.ac/institutes/', 'grin': 'http://www.ars.usda.gov/research/publications/publications.htm?seq_no_115=', 'grin.taxonomy': 'http://www.ars-grin.gov/cgi-bin/npgs/html/taxon.pl?', 'gro': 'http://www.bootstrep.eu/ontology/GRO#', 'grsdb': 'http://bioinformatics.ramapo.edu/GRSDB2/geneview.php?geneID=', 'gsfa': 'http://www.fao.org/gsfaonline/additives/details.html?id=', 'FAO/WHO_standards': 'http://www.fao.org/gsfaonline/additives/details.html?id=', 'GSSO': 'http://purl.obolibrary.org/obo/GSSO_', 'gtex': 'https://www.gtexportal.org/home/gene/', 'gtr': 'https://www.ncbi.nlm.nih.gov/gtr/conditions/', 'medgen.gtr': 'https://www.ncbi.nlm.nih.gov/gtr/conditions/', 'gudmap': 'https://gudmap.org/id/', 'gwascentral.marker': 'https://www.gwascentral.org/marker/', 'gwascentral.phenotype': 'https://www.gwascentral.org/phenotype/', 'gwascentral.study': 'https://www.gwascentral.org/study/', 'gxa.expt': 'https://www.ebi.ac.uk/gxa/experiments/', 'gxa.gene': 'https://www.ebi.ac.uk/gxa/genes/', 'HABRONATTUS': 'http://purl.obolibrary.org/obo/HABRONATTUS_', 'hagr.genage': 'http://genomics.senescence.info/genes/details.php?id=', 'hagr.gendr': 'http://genomics.senescence.info/diet/details.php?id=', 'hamap': 'https://hamap.expasy.org/unirule/', 'HANCESTRO': 'http://purl.obolibrary.org/obo/HANCESTRO_', 'HAO': 'http://purl.obolibrary.org/obo/HAO_', 'hathitrust': 'https://catalog.hathitrust.org/Record/', 'HBA': 'https://biopragmatics.github.io/providers/hba/', 'hbvar': 'http://globin.bx.psu.edu/cgi-bin/hbvar/query_vars3?mode=output&display_format=page&i=', 'hc.npn': 'http://webprod.hc-sc.gc.ca/nhpid-bdipsn/ingredReq.do?id=', 'hcpcs': 'http://purl.bioontology.org/ontology/HCPCS/', 'hcvdb': 'https://euhcvdb.ibcp.fr/euHCVdb/do/displayHCVEntry?primaryAC=', 'hdl': 'http://hdl.handle.net/', 'handle': 'http://hdl.handle.net/', 'hdr': 'http://research.nhgri.nih.gov/apps/homeodomain/web/index.cgi?mode=view&amp;view=proteins&amp;id=', 'HEPRO': 'http://purl.obolibrary.org/obo/HEPRO_', 'hgmd': 'http://www.hgmd.cf.ac.uk/ac/gene.php?gene=', 'hgnc': 'https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/', 'hgnc.genegroup': 'https://www.genenames.org/cgi-bin/genefamilies/set/', 'GFAM': 'https://www.genenames.org/cgi-bin/genefamilies/set/', 'HGNC_GROUP': 'https://www.genenames.org/cgi-bin/genefamilies/set/', 'hgnc.family': 'https://www.genenames.org/cgi-bin/genefamilies/set/', 'hgnc.genefamily': 'https://www.genenames.org/cgi-bin/genefamilies/set/', 'hgnc.symbol': 'https://www.genenames.org/cgi-bin/gene_symbol_report?match=', 'collagenmutdb': 'https://www.genenames.org/cgi-bin/gene_symbol_report?match=', 'depod': 'https://www.genenames.org/cgi-bin/gene_symbol_report?match=', 'pharmvar': 'https://www.genenames.org/cgi-bin/gene_symbol_report?match=', 'ygob': 'https://www.genenames.org/cgi-bin/gene_symbol_report?match=', 'hinv.locus': 'http://h-invitational.jp/hinv/spsoup/locus_view?hix_id=', 'hinv.protein': 'http://h-invitational.jp/hinv/protein/protein_view.cgi?hip_id=', 'hinv.transcript': 'http://h-invitational.jp/hinv/spsoup/transcript_view?hit_id=', 'hipsci': 'https://www.hipsci.org/lines/#/lines/', 'hl7.v2codesystem': 'http://terminology.hl7.org/CodeSystem/v2-', 'hl7.v3codesystem': 'https://terminology.hl7.org/CodeSystem-v3-', 'hmdb': 'http://www.hmdb.ca/metabolites/', 'HMDB': 'http://www.hmdb.ca/metabolites/', 'hms.lincs.antibody': 'https://lincs.hms.harvard.edu/db/antibodies/', 'hms.lincs.cell': 'https://lincs.hms.harvard.edu/db/cells/', 'hms.lincs.compound': 'https://lincs.hms.harvard.edu/db/sm/', 'HMS-LINCS': 'https://lincs.hms.harvard.edu/db/sm/', 'hmsl_id': 'https://lincs.hms.harvard.edu/db/sm/', 'hms.lincs.dataset': 'https://lincs.hms.harvard.edu/db/datasets/', 'hog': 'https://biopragmatics.github.io/providers/hog/', 'hogenom': 'http://pbil.univ-lyon1.fr/cgi-bin/view-tree.pl?db=HOGENOM5&query=', 'hoip': 'http://purl.bioontology.org/ontology/HOIP/HOIP_', 'HOM': 'http://purl.obolibrary.org/obo/HOM_', 'homd.seq': 'http://www.homd.org/modules.php?op=modload&name=GenomeList&file=index&link=detailinfo&seqid=', 'homd.taxon': 'http://www.homd.org/modules.php?op=modload&name=HOMD&view=dynamic&oraltaxonid=', 'homologene': 'https://www.ncbi.nlm.nih.gov/homologene/', 'horizon_discovery': 'https://horizondiscovery.com/en/search?searchterm=', 'hovergen': 'https://identifiers.org/hovergen:', 'HP': 'http://purl.obolibrary.org/obo/HP_', 'hpo': 'http://purl.obolibrary.org/obo/HP_', 'hpa': 'http://www.proteinatlas.org/', 'hpath': 'http://purl.obolibrary.org/obo/MC_', 'MC': 'http://purl.obolibrary.org/obo/MC_', 'hpm.peptide': 'http://www.humanproteomemap.org/spectrum.php?pep_id=', 'hpm.protein': 'http://www.humanproteomemap.org/protein.php?hpm_id=', 'hprd': 'http://www.hprd.org/protein/', 'hpscreg': 'https://hpscreg.eu/cell-line/', 'HsapDv': 'http://purl.obolibrary.org/obo/HsapDv_', 'hsdb': 'https://pubchem.ncbi.nlm.nih.gov/source/hsdb/', 'HSO': 'http://purl.obolibrary.org/obo/HSO_', 'hssp': 'https://identifiers.org/hssp:', 'HTN': 'http://purl.obolibrary.org/obo/HTN_', 'huge': 'https://www.kazusa.or.jp/huge/gfpage/', 'IAO': 'http://purl.obolibrary.org/obo/IAO_', 'ICD10WHO': 'https://icd.who.int/browse10/2019/en#/', 'ICD': 'https://icd.who.int/browse10/2019/en#/', 'ICD-10': 'https://icd.who.int/browse10/2019/en#/', 'ICD10': 'https://icd.who.int/browse10/2019/en#/', 'icd10cm': 'https://icd.codes/icd10cm/', 'ICD10CM': 'https://icd.codes/icd10cm/', 'icd10pcs': 'https://www.findacode.com/code.php?set=ICD10PCS&c=', 'icd11': 'https://icd.who.int/browse11/l-m/en#/http://id.who.int/icd/entity/', 'ICD-11': 'https://icd.who.int/browse11/l-m/en#/http://id.who.int/icd/entity/', 'icd9cm': 'http://icd9cm.chrisendres.com/index.php?action=search&srchtext=', 'IC9CM': 'http://icd9cm.chrisendres.com/index.php?action=search&srchtext=', 'ICD9CM': 'http://icd9cm.chrisendres.com/index.php?action=search&srchtext=', 'ICD9CM_2005': 'http://icd9cm.chrisendres.com/index.php?action=search&srchtext=', 'ICD9CM_2006': 'http://icd9cm.chrisendres.com/index.php?action=search&srchtext=', 'ICDCM_2005': 'http://icd9cm.chrisendres.com/index.php?action=search&srchtext=', 'icdc': 'https://caninecommons.cancer.gov/#/study/', 'icdo': 'http://codes.iarc.fr/search.php?cx=009987501641899931167%3A2_7lsevqpdm&cof=FORID%3A9&ie=UTF-8&ie=ISO-8859-1&oe=ISO-8859-1&sa=&q=', 'ICD-O': 'http://codes.iarc.fr/search.php?cx=009987501641899931167%3A2_7lsevqpdm&cof=FORID%3A9&ie=UTF-8&ie=ISO-8859-1&oe=ISO-8859-1&sa=&q=', 'ICDO': 'http://codes.iarc.fr/search.php?cx=009987501641899931167%3A2_7lsevqpdm&cof=FORID%3A9&ie=UTF-8&ie=ISO-8859-1&oe=ISO-8859-1&sa=&q=', 'iceberg.element': 'https://bioinfo-mml.sjtu.edu.cn/ICEberg2/feature_page.php?ice_id=', 'iceberg.ice': 'https://bioinfo-mml.sjtu.edu.cn/ICEberg2/feature_page.php?ice_id=', 'iceberg.family': 'https://bioinfo-mml.sjtu.edu.cn/ICEberg2/browse_result.php?type=fam&fam_id=', 'ICEO': 'http://purl.obolibrary.org/obo/ICEO_', 'icepo': 'https://biopragmatics.github.io/providers/icepo/', 'icf': 'http://id.who.int/icd/entity/', 'iclc': 'http://www.iclc.it/details/det_list.php?line_id=', 'icldb': 'https://entomology.ca.uky.edu/content/', 'ICO': 'http://purl.obolibrary.org/obo/ICO_', 'ideal': 'http://idp1.force.cs.is.nagoya-u.ac.jp/IDEAL/ideal.php?id=', 'IDO': 'http://purl.obolibrary.org/obo/IDO_', 'idocovid19': 'http://purl.obolibrary.org/obo/COVIDO_', 'idoden': 'http://purl.bioontology.org/ontology/IDODEN_', 'idog': 'https://ngdc.cncb.ac.cn/idog/breed/getBreedDetail.action?breedId=', 'IDOMAL': 'http://purl.obolibrary.org/obo/IDOMAL_', 'idoo': 'http://registry.api.hq.identifiers.org/semanticApi/getRegistryOntology#', 'idot': 'https://biomodels.net/vocab/idot.rdf#', 'idpo': 'https://www.disprot.org/idpo/IDPO:', 'idr': 'https://idr.openmicroscopy.org/search/?query=Name:idr', 'iedb': 'https://www.iedb.org/reference/', 'IEV': 'http://purl.obolibrary.org/obo/IEV_', 'igrhcellid': 'http://igrcid.ibms.sinica.edu.tw/cgi-bin/cell_line_view.cgi?cl_name=', 'igsn': 'http://igsn.org/', 'igsr': 'https://www.internationalgenome.org/data-portal/sample/', 'imanis': 'https://www.imanislife.com/?s=', 'imex': 'https://www.ebi.ac.uk/intact/imex/main.xhtml?query=IM-', 'img.gene': 'http://img.jgi.doe.gov/cgi-bin/w/main.cgi?section=GeneDetail&gene_oid=', 'img.taxon': 'http://img.jgi.doe.gov/cgi-bin/w/main.cgi?section=TaxonDetail&taxon_oid=', 'imgt.hla': 'https://www.ebi.ac.uk/cgi-bin/imgt/hla/get_allele.cgi?', 'IPD-IMGT/HLA': 'https://www.ebi.ac.uk/cgi-bin/imgt/hla/get_allele.cgi?', 'imgt.ligm': 'http://www.imgt.org/ligmdb/view?id=', 'imotdb': 'http://caps.ncbs.res.in/cgi-bin/mini/databases/imotdb/imotdb.cgi?sfcode=', 'IMR': 'http://purl.obolibrary.org/obo/IMR_', 'inaturalist.observation': 'https://www.inaturalist.org/observations/', 'inaturalist.place': 'https://www.inaturalist.org/places/', 'inaturalist.taxon': 'https://www.inaturalist.org/taxa/', 'inaturalist.user': 'https://www.inaturalist.org/users/', 'inchi': 'http://www.chemspider.com/', 'inchikey': 'http://www.chemspider.com/inchikey=', 'inn': 'https://mednet-communities.net/inn/db/ViewINN.aspx?i=', 'INN_ID': 'https://mednet-communities.net/inn/db/ViewINN.aspx?i=', 'innatedb': 'http://www.innatedb.ca/getGeneCard.do?id=', 'INO': 'http://purl.obolibrary.org/obo/INO_', 'insdc': 'https://www.ncbi.nlm.nih.gov/nuccore/', 'nucleotide': 'https://www.ncbi.nlm.nih.gov/nuccore/', 'insdc.sra': 'https://www.ncbi.nlm.nih.gov/sra/', 'intact': 'https://www.ebi.ac.uk/intact/interaction/', 'intact.molecule': 'https://www.ebi.ac.uk/intact/search?query=', 'interfil': 'http://www.interfil.org/details.php?id=', 'interlex': 'https://scicrunch.org/scicrunch/interlex/view/ilx_', 'ILX': 'https://scicrunch.org/scicrunch/interlex/view/ilx_', 'IPR': 'http://purl.obolibrary.org/obo/IPR_', 'IP': 'http://purl.obolibrary.org/obo/IPR_', 'InterPro': 'http://purl.obolibrary.org/obo/IPR_', 'iobc': 'http://purl.jp/bio/4/id/', 'ird.segment': 'http://www.fludb.org/brc/fluSegmentDetails.do?ncbiGenomicAccession=', 'irefweb': 'http://wodaklab.org/iRefWeb/interaction/show/', 'iresite': 'http://www.iresite.org/IRESite_web.php?page=view&entry_id=', 'iro': 'https://legacy.vectorbase.org/ontology-browser?cv=MIRO&t=IRO:', 'isbn': 'http://isbndb.com/search-all.html?kw=', 'ISBN-10': 'http://isbndb.com/search-all.html?kw=', 'ISBN-13': 'http://isbndb.com/search-all.html?kw=', 'isfinder': 'https://www-is.biotoul.fr/scripts/ficheIS.php?name=', 'isni': 'http://www.isni.org/isni/', 'iso15926': 'http://standards.iso.org/iso/15926/part14/', 'isrctn': 'https://www.isrctn.com/', 'issn': 'https://portal.issn.org/resource/ISSN/', 'itis': 'https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=', 'ito': 'https://bioportal.bioontology.org/ontologies/ITO/?p=classes&conceptid=https://identifiers.org/ito:ITO_', 'iuphar.family': 'http://www.guidetopharmacology.org/GRAC/FamilyDisplayForward?familyId=', 'IUPHARfam': 'http://www.guidetopharmacology.org/GRAC/FamilyDisplayForward?familyId=', 'iuphar.ligand': 'http://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=', 'IUPHAR_LIGAND': 'http://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=', 'IUPHAR_LIGAND_ID': 'http://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=', 'iuphar.receptor': 'http://www.guidetopharmacology.org/GRAC/ObjectDisplayForward?objectId=', 'IUPHAR_GPCR': 'http://www.guidetopharmacology.org/GRAC/ObjectDisplayForward?objectId=', 'IUPHARobj': 'http://www.guidetopharmacology.org/GRAC/ObjectDisplayForward?objectId=', 'ivdb': 'http://influenza.psych.ac.cn/search/detail.jsp?segment.gb_id=', 'jax': 'https://www.jax.org/strain/', 'jaxmice': 'https://identifiers.org/jaxmice:', 'jcggdb': 'http://jcggdb.jp/idb/jcggdb/', 'jcm': 'http://www.jcm.riken.go.jp/cgi-bin/jcm/jcm_number?JCM=', 'jcrb': 'https://cellbank.nibiohn.go.jp/~cellbank/en/search_res_det.cgi?RNO=', 'jcsd': 'http://jglobal.jst.go.jp/en/redirect?Nikkaji_No=', 'jgi.proposal': 'https://genome.jgi.doe.gov/portal/lookup?keyName=proposalId&groupOnly=1&app=Info&keyValue=', 'jrct': 'https://jrct.niph.go.jp/en-latest-detail/', 'jstor': 'http://www.jstor.org/stable/', 'jws': 'https://jjj.bio.vu.nl/models/', 'kaggle': 'https://www.kaggle.com/', 'kclb': 'https://cellbank.snu.ac.kr/english/sub/catalog.php?s_cellid=464&page=detail_info&CatNo=59&strQ=', 'kegg': 'http://www.kegg.jp/entry/', 'KEGG': 'http://www.kegg.jp/entry/', 'KEGG COMPOUND': 'http://www.kegg.jp/entry/', 'KEGG DRUG': 'http://www.kegg.jp/entry/', 'KEGG-path': 'http://www.kegg.jp/entry/', 'KEGG.COMPOUND': 'http://www.kegg.jp/entry/', 'KEGG.DRUG': 'http://www.kegg.jp/entry/', 'KEGG.EDRUG': 'http://www.kegg.jp/entry/', 'KEGG_PATHWAY': 'http://www.kegg.jp/entry/', 'KEGG_REACTION': 'http://www.kegg.jp/entry/', 'kegg.compound': 'http://www.kegg.jp/entry/', 'kegg.disease': 'http://www.kegg.jp/entry/', 'kegg.drug': 'http://www.kegg.jp/entry/', 'kegg.environ': 'http://www.kegg.jp/entry/', 'kegg.genes': 'http://www.kegg.jp/entry/', 'kegg.genome': 'http://www.kegg.jp/entry/', 'kegg.glycan': 'http://www.kegg.jp/entry/', 'kegg.ligand': 'http://www.kegg.jp/entry/', 'kegg.metagenome': 'http://www.kegg.jp/entry/', 'kegg.module': 'http://www.kegg.jp/entry/', 'kegg.orthology': 'http://www.kegg.jp/entry/', 'kegg.pathway': 'http://www.kegg.jp/entry/', 'kegg.reaction': 'http://www.kegg.jp/entry/', 'kegg_genome': 'http://www.kegg.jp/entry/', 'kegg_genomes': 'http://www.kegg.jp/entry/', 'kegg.brite': 'http://www.genome.jp/kegg-bin/get_htext?-p+/kegg/brite/br+br', 'kestrelo': 'http://purl.obolibrary.org/obo/kestrelo_', 'knapsack': 'http://www.knapsackfamily.com/knapsack_core/information.php?word=', 'LABO': 'http://purl.obolibrary.org/obo/LABO_', 'langual': 'https://www.langual.org/langual_thesaurus.asp?termid=', 'lbo': 'http://purl.obolibrary.org/obo/LBO_', 'lcsh': 'http://id.loc.gov/authorities/names/', 'lcnaf': 'http://id.loc.gov/authorities/names/', 'lcnafid': 'http://id.loc.gov/authorities/names/', 'lei': 'https://www.gleif.org/lei/', 'LEPAO': 'http://purl.obolibrary.org/obo/LEPAO_', 'lgai.cede': 'https://s3.us-east-2.amazonaws.com/lg.cede/', 'lgic': 'https://identifiers.org/lgic:', 'licebase': 'https://licebase.org/?q=', 'ligandbook': 'https://ligandbook.org/package/', 'ligandbox': 'http://www.mypresto5.com/ligandbox/cgi-bin/liginf.cgi?id=', 'ligandexpo': 'https://identifiers.org/ligandexpo:', 'ligea': 'http://hpc-bioinformatics.cineca.it/fusion/cell_line/', 'limore': 'https://www.picb.ac.cn/limore/cellLines/single?para=', 'lincs.cell': 'http://lincsportal.ccs.miami.edu/cells/#/view/', 'LINCS_LDP': 'http://lincsportal.ccs.miami.edu/cells/#/view/', 'lincs.data': 'http://lincsportal.ccs.miami.edu/datasets/#/view/', 'lincs.protein': 'https://lincs.hms.harvard.edu/db/proteins/', 'lincs.smallmolecule': 'http://lincsportal.ccs.miami.edu/SmallMolecules/#/view/', 'lincs': 'http://lincsportal.ccs.miami.edu/SmallMolecules/#/view/', 'linguist': 'https://raw.githubusercontent.com/github/linguist/master/lib/linguist/languages.yml#', 'linkml': 'https://w3id.org/linkml/', 'lipidbank': 'http://lipidbank.jp/cgi-bin/detail.cgi?id=', 'lipidmaps': 'http://www.lipidmaps.org/data/LMSDRecord.php?LMID=', 'LIPID MAPS': 'http://www.lipidmaps.org/data/LMSDRecord.php?LMID=', 'LIPID_MAPS_class': 'http://www.lipidmaps.org/data/LMSDRecord.php?LMID=', 'LIPID_MAPS_instance': 'http://www.lipidmaps.org/data/LMSDRecord.php?LMID=', 'LIPRO': 'http://purl.obolibrary.org/obo/LIPRO_', 'LOGGERHEAD': 'http://purl.obolibrary.org/obo/LOGGERHEAD_', 'loinc': 'http://loinc.org/rdf/', 'LNC': 'http://loinc.org/rdf/', 'lonza': 'https://knowledge.lonza.com/cell?id=', 'loqate': 'http://www.weizmann.ac.il/molgen/loqate/gene/view/', 'lotus': 'https://lotus.naturalproducts.net/compound/lotus_id/', 'lpt': 'http://purl.obolibrary.org/obo/LPT_', 'lrg': 'https://identifiers.org/lrg:', 'lspci': 'https://labsyspharm.github.io/lspci/', 'lter': 'https://vocab.lternet.edu/vocab/vocab/index.php?tema=', 'MA': 'http://purl.obolibrary.org/obo/MA_', 'macie': 'https://www.ebi.ac.uk/thornton-srv/databases/cgi-bin/MACiE/entry/getPage.pl?id=', 'maizegdb.locus': 'http://www.maizegdb.org/cgi-bin/displaylocusrecord.cgi?id=', 'MaizeGDB': 'http://www.maizegdb.org/cgi-bin/displaylocusrecord.cgi?id=', 'MAMO': 'http://purl.obolibrary.org/obo/MAMO_', 'mampol': 'http://mampol.uab.es/cgi-bin/MamPol_s2.pl?Accession=', 'MAO': 'http://purl.obolibrary.org/obo/MAO_', 'massbank': 'https://massbank.jp/RecordDisplay?id=', 'massive': 'https://massive.ucsd.edu/ProteoSAFe/QueryMSV?id=', 'MAT': 'http://purl.obolibrary.org/obo/MAT_', 'matrixdb': 'http://matrixdb.univ-lyon1.fr/cgi-bin/current/newPort?type=biomolecule&value=', 'matrixdb.association': 'https://identifiers.org/matrixdb.association:', 'MAXO': 'http://purl.obolibrary.org/obo/MAXO_', 'MBA': 'https://biopragmatics.github.io/providers/mba/', 'mcc': 'http://www.semanticweb.org/pallabi.d/ontologies/2014/2/untitled-ontology-11#', 'MCO': 'http://purl.obolibrary.org/obo/MCO_', 'mdm': 'https://medical-data-models.org/forms/', 'meddra': 'http://purl.bioontology.org/ontology/MDRFRE/', 'MEDRA': 'http://purl.bioontology.org/ontology/MDRFRE/', 'MeDRA': 'http://purl.bioontology.org/ontology/MDRFRE/', 'MedDRA': 'http://purl.bioontology.org/ontology/MDRFRE/', 'Medical Dictionary for Regulatory Activities': 'http://purl.bioontology.org/ontology/MDRFRE/', 'medgen': 'https://www.ncbi.nlm.nih.gov/medgen/', 'medlineplus': 'https://identifiers.org/medlineplus:', 'merops.clan': 'https://www.ebi.ac.uk/merops/cgi-bin/clansum?clan=', 'merops.entry': 'https://www.ebi.ac.uk/merops/cgi-bin/pepsum?id=', 'merops.inhibitor': 'https://www.ebi.ac.uk/merops/cgi-bin/pepsum?id=', 'merops.family': 'https://www.ebi.ac.uk/merops/cgi-bin/famsum?family=', 'mesh': 'http://id.nlm.nih.gov/mesh/', 'MESH': 'http://id.nlm.nih.gov/mesh/', 'MESHA': 'http://id.nlm.nih.gov/mesh/', 'MESHC': 'http://id.nlm.nih.gov/mesh/', 'MESHCS': 'http://id.nlm.nih.gov/mesh/', 'MESHD': 'http://id.nlm.nih.gov/mesh/', 'MESHPP': 'http://id.nlm.nih.gov/mesh/', 'MESH_DESCRIPTOR_UI': 'http://id.nlm.nih.gov/mesh/', 'MESH_SUPPLEMENTAL_RECORD_UI': 'http://id.nlm.nih.gov/mesh/', 'MSH': 'http://id.nlm.nih.gov/mesh/', 'MSH2010_2010_02_22': 'http://id.nlm.nih.gov/mesh/', 'ctd.chemical': 'http://id.nlm.nih.gov/mesh/', 'ctd.disease': 'http://id.nlm.nih.gov/mesh/', 'mesh.2012': 'http://id.nlm.nih.gov/mesh/', 'mesh.2013': 'http://id.nlm.nih.gov/mesh/', 'mesh.vocab': 'http://id.nlm.nih.gov/mesh/vocab#', 'metabolights': 'https://www.ebi.ac.uk/metabolights/', 'metacyc.compound': 'https://metacyc.org/compound?orgid=META&id=', 'MetaCyc': 'https://metacyc.org/compound?orgid=META&id=', 'metacyc.reaction': 'https://metacyc.org/META/NEW-IMAGE?type=REACTION&object=', 'metanetx.chemical': 'https://www.metanetx.org/chem_info/', 'metanetx.compartment': 'https://www.metanetx.org/comp_info/', 'metanetx.reaction': 'https://www.metanetx.org/equa_info/', 'metatlas': 'https://metabolicatlas.org/identifier/MetabolicAtlas/', 'metatlas.metabolite': 'https://metabolicatlas.org/identifier/MetabolicAtlas/', 'metatlas.reaction': 'https://metabolicatlas.org/identifier/MetabolicAtlas/', 'metlin': 'http://metlin.scripps.edu/metabo_info.php?molid=', 'metnetdb': 'http://metnetonline.org/browse_pathway2.php?pthID=', 'mex': 'https://www.metabolome-express.org/datasetview.php?datasetid=', 'MF': 'http://purl.obolibrary.org/obo/MF_', 'MFMO': 'http://purl.obolibrary.org/obo/MFMO_', 'MFO': 'http://purl.obolibrary.org/obo/MFO_', 'MFOEM': 'http://purl.obolibrary.org/obo/MFOEM_', 'MFOMD': 'http://purl.obolibrary.org/obo/MFOMD_', 'mge': 'http://aclame.ulb.ac.be/perl/Aclame/Genomes/mge_view.cgi?view=info&id=mge:', 'aclame': 'http://aclame.ulb.ac.be/perl/Aclame/Genomes/mge_view.cgi?view=info&id=mge:', 'MGI': 'http://www.informatics.jax.org/accession/MGI:', 'MGD': 'http://www.informatics.jax.org/accession/MGI:', 'mgnify.analysis': 'https://www.ebi.ac.uk/metagenomics/analyses/', 'mgnify.proj': 'https://www.ebi.ac.uk/metagenomics/projects/', 'mgnify.samp': 'https://www.ebi.ac.uk/metagenomics/samples/', 'MI': 'http://purl.obolibrary.org/obo/MI_', 'PSI-MI': 'http://purl.obolibrary.org/obo/MI_', 'MIAPA': 'http://purl.obolibrary.org/obo/MIAPA_', 'MICRO': 'http://purl.obolibrary.org/obo/MICRO_', 'microscope': 'http://www.genoscope.cns.fr/agc/microscope/mage/info.php?id=', 'microsporidia': 'http://microsporidiadb.org/micro/showRecord.do?name=GeneRecordClasses.GeneRecordClass&source_id=', 'millipore': 'https://www.merckmillipore.com/catalogue/item/', 'mimodb': 'http://immunet.cn/bdb/index.php/mimoset/', 'minid': 'https://hdl.handle.net/hdl:20.500.12582/', 'minid.test': 'https://hdl.handle.net/hdl:20.500.12633/', 'mint': 'https://mint.bio.uniroma2.it/index.php/detailed-curation/?id=MINT-', 'mipmod': 'http://bioinfo.iitk.ac.in/MIPModDB/result.php?code=', 'mir': 'https://registry.identifiers.org/registry?query=MIR:', 'mirbase': 'http://www.mirbase.org/cgi-bin/mirna_entry.pl?acc=', 'mirbase.family': 'http://www.mirbase.org/cgi-bin/mirna_summary.pl?fam=', 'MIRBASE_FAMILY': 'http://www.mirbase.org/cgi-bin/mirna_summary.pl?fam=', 'mirbase.mature': 'http://mirbase.org/cgi-bin/mature.pl?mature_acc=', 'MIRBASEM': 'http://mirbase.org/cgi-bin/mature.pl?mature_acc=', 'mirex': 'https://identifiers.org/mirex:', 'miriam': 'https://registry.identifiers.org/registry/', 'identifiers.namespace': 'https://registry.identifiers.org/registry/', 'miriam.collection': 'https://www.ebi.ac.uk/miriam/main/', 'miriam.resource': 'https://www.ebi.ac.uk/miriam/main/resources/', 'MIRNAO': 'http://purl.obolibrary.org/obo/MIRNAO_', 'mirnest': 'http://rhesus.amu.edu.pl/mirnest/copy/details.php?id=', 'MIRO': 'http://purl.obolibrary.org/obo/MIRO_', 'mirtarbase': 'http://mirtarbase.mbc.nctu.edu.tw/php/detail.php?mirtid=', 'mixs': 'https://w3id.org/mixs/', 'mlc': 'https://www.mlcommons.org/mlc-id/', 'mmdb': 'http://www.ncbi.nlm.nih.gov/Structure/mmdb/mmdbsrv.cgi?uid=', 'mmmp.biomaps': 'http://www.mmmp.org/MMMP/public/biomap/viewBiomap.mmmp?id=', 'mmmp:biomaps': 'http://www.mmmp.org/MMMP/public/biomap/viewBiomap.mmmp?id=', 'MMO': 'http://purl.obolibrary.org/obo/MMO_', 'mmp.cat': 'https://mmp.sfb.uit.no/databases/marcat/#/records/', 'mmp.db': 'https://mmp.sfb.uit.no/databases/mardb/#/records/', 'mmp.fun': 'https://mmp.sfb.uit.no/databases/marfun/#/records/', 'mmp.ref': 'https://mmp.sfb.uit.no/databases/marref/#/records/', 'mmrrc': 'http://www.mmrrc.org/catalog/getSDS.php?mmrrc_id=', 'mmsinc': 'http://mms.dsfarm.unipd.it/mmsinc/search/molecule.php?mmscode=', 'MmusDv': 'http://purl.obolibrary.org/obo/MmusDv_', 'MO': 'http://purl.obolibrary.org/obo/MO_', 'MOD': 'http://purl.obolibrary.org/obo/MOD_', 'PSI-MOD': 'http://purl.obolibrary.org/obo/MOD_', 'modeldb': 'https://modeldb.science/', 'modeldb.concept': 'https://senselab.med.yale.edu/ModelDB/ModelList?id=', 'molbase': 'http://www.molbase.com/en/index.php?app=search&search_keyword=', 'molmedb': 'https://molmedb.upol.cz/mol/', 'molmovdb': 'http://www.molmovdb.org/cgi-bin/morph.cgi?ID=', 'MONDO': 'http://purl.obolibrary.org/obo/MONDO_', 'MOP': 'http://purl.obolibrary.org/obo/MOP_', 'morpheus': 'https://morpheus.gitlab.io/models/', 'mosaic': 'https://purl.dataone.org/odo/MOSAIC_', 'MP': 'http://purl.obolibrary.org/obo/MP_', 'MPATH': 'http://purl.obolibrary.org/obo/MPATH_', 'mpid': 'http://www.jcvi.org/mpidb/experiment.php?interaction_id=', 'mpidb': 'http://www.jcvi.org/mpidb/experiment.php?interaction_id=', 'MPIO': 'http://purl.obolibrary.org/obo/MPIO_', 'MRO': 'http://purl.obolibrary.org/obo/MRO_', 'MS': 'http://purl.obolibrary.org/obo/MS_', 'msigdb': 'https://biopragmatics.github.io/providers/msigdb/', 'msig': 'https://biopragmatics.github.io/providers/msigdb/', 'msio': 'http://purl.obolibrary.org/obo/MSIO_', 'mtbd': 'http://tumor.informatics.jax.org/mtbwi/strainDetails.do?key=', 'multicellds': 'http://multicellds.org/MultiCellDB/', 'multicellds.cell_line': 'http://multicellds.org/MultiCellDB/', 'multicellds.collection': 'http://multicellds.org/MultiCellDB/', 'multicellds.snapshot': 'http://multicellds.org/MultiCellDB/', 'mw.project': 'http://www.metabolomicsworkbench.org/data/DRCCMetadata.php?Mode=Project&ProjectID=', 'mw.study': 'http://www.metabolomicsworkbench.org/data/DRCCMetadata.php?Mode=Study&StudyID=', 'myco.lepra': 'http://mycobrowser.epfl.ch/leprosysearch.php?gene+name=', 'myco.marinum': 'http://mycobrowser.epfl.ch/marinosearch.php?gene+name=', 'myco.smeg': 'http://mycobrowser.epfl.ch/smegmasearch.php?gene+name=', 'myco.tuber': 'http://tuberculist.epfl.ch/quicksearch.php?gene+name=', 'TubercuList': 'http://tuberculist.epfl.ch/quicksearch.php?gene+name=', 'mycobank': 'http://www.mycobank.org/Biolomics.aspx?Table=Mycobank&MycoBankNr_=', 'mzspec': 'http://proteomecentral.proteomexchange.org/usi/?usi=mzspec:', 'nando': 'https://nanbyodata.jp/disease/NANDO:', 'napdi': 'https://repo.napdi.org/study/', 'napp': 'http://rna.igmors.u-psud.fr/NAPP/Niveau2.php?specie=', 'narcis': 'http://www.narcis.nl/publication/RecordID/', 'nasc': 'http://arabidopsis.info/StockInfo?NASC_id=', 'nbn': 'https://identifiers.org/nbn:', 'NBO': 'http://purl.obolibrary.org/obo/NBO_', 'nbrc': 'http://www.nbrc.nite.go.jp/NBRC2/NBRCCatalogueDetailServlet?ID=NBRC&CAT=', 'ncats.bioplanet': 'https://tripod.nih.gov/bioplanet/detail.jsp?pid=bioplanet_', 'ncats.drug': 'https://drugs.ncats.io/drug/', 'inxight': 'https://drugs.ncats.io/drug/', 'inxight-drugs': 'https://drugs.ncats.io/drug/', 'ncbi.assembly': 'https://www.ncbi.nlm.nih.gov/datasets/genome/', 'insdc.gca': 'https://www.ncbi.nlm.nih.gov/datasets/genome/', 'ncbi.genome': 'https://www.ncbi.nlm.nih.gov/genome/', 'ncbi.resource': 'https://bioregistry.io/metaregistry/ncbi/', 'ncbibook': 'https://www.ncbi.nlm.nih.gov/books/', 'NCBIGene': 'http://purl.uniprot.org/geneid/', 'EGID': 'http://purl.uniprot.org/geneid/', 'EntrezGene': 'http://purl.uniprot.org/geneid/', 'NCBI_GeneID': 'http://purl.uniprot.org/geneid/', 'ctd.gene': 'http://purl.uniprot.org/geneid/', 'entrez': 'http://purl.uniprot.org/geneid/', 'entrez gene/locuslink': 'http://purl.uniprot.org/geneid/', 'genewiki': 'http://purl.uniprot.org/geneid/', 'wikigenes': 'http://purl.uniprot.org/geneid/', 'NCBIProtein': 'https://www.ncbi.nlm.nih.gov/protein/', 'REFSEQ_PROT': 'https://www.ncbi.nlm.nih.gov/protein/', 'insdc.cds': 'https://www.ncbi.nlm.nih.gov/protein/', 'refseq': 'https://www.ncbi.nlm.nih.gov/protein/', 'NCBITaxon': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'NCBI Taxonomy': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'NCBITaxonomyID': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'NCBI_Taxon_ID': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'NCBI_taxid': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'TAX': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'TaxonomyID': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'cutg': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'taxid': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'taxon': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'taxonomy': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'uniprot.taxonomy': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'virmirdb': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'virushostdb': 'http://purl.obolibrary.org/obo/NCBITaxon_', 'ncim': 'http://ncim.nci.nih.gov/ncimbrowser/ConceptReport.jsp?dictionary=NCI%20MetaThesaurus&code=', 'NCIT': 'http://purl.obolibrary.org/obo/NCIT_', 'NCI': 'http://purl.obolibrary.org/obo/NCIT_', 'NCI2004_11_17': 'http://purl.obolibrary.org/obo/NCIT_', 'NCIT_Thesaurus': 'http://purl.obolibrary.org/obo/NCIT_', 'NCITm': 'http://purl.obolibrary.org/obo/NCIT_', 'NCITt': 'http://purl.obolibrary.org/obo/NCIT_', 'NCI_Thesaurus': 'http://purl.obolibrary.org/obo/NCIT_', 'NCIt': 'http://purl.obolibrary.org/obo/NCIT_', 'NCit': 'http://purl.obolibrary.org/obo/NCIT_', 'ncithesaurus': 'http://purl.obolibrary.org/obo/NCIT_', 'NCRO': 'http://purl.obolibrary.org/obo/NCRO_', 'ndc': 'http://www.hipaaspace.com/Medical_Billing/Coding/National.Drug.Codes/', 'nddf': 'http://purl.bioontology.org/ontology/NDDF/', 'ndex': 'https://www.ndexbio.org/viewer/networks/', 'ndfrt': 'http://purl.bioontology.org/ontology/NDFRT/', 'NUI': 'http://purl.bioontology.org/ontology/NDFRT/', 'ndf-rt': 'http://purl.bioontology.org/ontology/NDFRT/', 'nembase': 'http://www.nematodes.org/nembase4/cluster.php?cluster=', 'nemo': 'https://assets.nemoarchive.org/', 'nemo2': 'http://purl.bioontology.org/NEMO/ontology/NEMO.owl#NEMO_', 'neurolex': 'http://uri.neuinfo.org/nif/nifstd/nlx_', 'NLX': 'http://uri.neuinfo.org/nif/nifstd/nlx_', 'nlx': 'http://uri.neuinfo.org/nif/nifstd/nlx_', 'neuromorpho': 'http://neuromorpho.org/neuron_info.jsp?neuron_name=', 'neuronames': 'http://braininfo.rprc.washington.edu/centraldirectory.aspx?ID=', 'neuroname': 'http://braininfo.rprc.washington.edu/centraldirectory.aspx?ID=', 'neurondb': 'http://senselab.med.yale.edu/NeuronDB/NeuronProp.aspx?id=', 'neurovault.collection': 'https://neurovault.org/collections/', 'neurovault.image': 'https://neurovault.org/images/', 'nextdb': 'http://nematode.lab.nig.ac.jp/db2/ShowCloneInfo.php?clone=', 'nextprot': 'https://www.nextprot.org/db/entry/', 'NXP': 'https://www.nextprot.org/db/entry/', 'nextprot.family': 'https://www.nextprot.org/term/FA-', 'NXPFA': 'https://www.nextprot.org/term/FA-', 'nfdi4chem.cao': 'http://champ-project.org/images/ontology/cao.owl#CAO_', 'NGBO': 'http://purl.obolibrary.org/obo/NGBO_', 'ngl': 'https://genelab-data.ndc.nasa.gov/genelab/accession/', 'nhcdr': 'https://stemcells.nindsgenetics.org?line=', 'niaest': 'http://lgsun.grc.nia.nih.gov/cgi-bin/pro3?sname1=', 'NIF_CELL': 'http://purl.obolibrary.org/obo/NIF_CELL_', 'NIF_DYSFUNCTION': 'http://purl.obolibrary.org/obo/NIF_DYSFUNCTION_', 'nif.ext': 'http://uri.neuinfo.org/nif/nifstd/nifext_', 'NIFEXT': 'http://uri.neuinfo.org/nif/nifstd/nifext_', 'NIF_GROSSANATOMY': 'http://purl.obolibrary.org/obo/NIF_GROSSANATOMY_', 'nif.std': 'http://uri.neuinfo.org/nif/nifstd/', 'NIFSTD': 'http://uri.neuinfo.org/nif/nifstd/', 'nihhesc': 'https://grants.nih.gov/stem_cells/registry/current.htm?ID=', 'nihreporter.project': 'https://reporter.nih.gov/project-details/', 'nkos': 'http://w3id.org/nkos/', 'nlfff': 'http://database.deepsolar.space:18080/dbs/nlfff/', 'nlm': 'https://www.ncbi.nlm.nih.gov/nlmcatalog/', 'nlx.anat': 'http://uri.neuinfo.org/nif/nifstd/nlx_anat_', 'nlx.br': 'http://uri.neuinfo.org/nif/nifstd/nlx_br_', 'NLXBR': 'http://uri.neuinfo.org/nif/nifstd/nlx_br_', 'nlx.cell': 'http://uri.neuinfo.org/nif/nifstd/nlx_cell_', 'NLXCELL': 'http://uri.neuinfo.org/nif/nifstd/nlx_cell_', 'nlx.chem': 'http://uri.neuinfo.org/nif/nifstd/nlx_chem_', 'NLXCHEM': 'http://uri.neuinfo.org/nif/nifstd/nlx_chem_', 'nlx.dys': 'http://uri.neuinfo.org/nif/nifstd/nlx_dys_', 'nlx.func': 'http://uri.neuinfo.org/nif/nifstd/nlx_func_', 'NLXFUNC': 'http://uri.neuinfo.org/nif/nifstd/nlx_func_', 'nlx.inv': 'http://uri.neuinfo.org/nif/nifstd/nlx_inv_', 'NLXINV': 'http://uri.neuinfo.org/nif/nifstd/nlx_inv_', 'nlx.mol': 'http://uri.neuinfo.org/nif/nifstd/nlx_mol_', 'NLXMOL': 'http://uri.neuinfo.org/nif/nifstd/nlx_mol_', 'nlx.oen': 'http://uri.neuinfo.org/nif/nifstd/oen_', 'NLXOEN': 'http://uri.neuinfo.org/nif/nifstd/oen_', 'nlx.org': 'http://uri.neuinfo.org/nif/nifstd/nlx_organ_', 'NLXORG': 'http://uri.neuinfo.org/nif/nifstd/nlx_organ_', 'nlx.qual': 'http://uri.neuinfo.org/nif/nifstd/nlx_qual_', 'NLXQUAL': 'http://uri.neuinfo.org/nif/nifstd/nlx_qual_', 'nlx.res': 'http://uri.neuinfo.org/nif/nifstd/nlx_res_', 'NLXRES': 'http://uri.neuinfo.org/nif/nifstd/nlx_res_', 'nlx.sub': 'http://uri.neuinfo.org/nif/nifstd/nlx_subcell_', 'NLXSUB': 'http://uri.neuinfo.org/nif/nifstd/nlx_subcell_', 'nif_subcellular': 'http://uri.neuinfo.org/nif/nifstd/nlx_subcell_', 'nmdc': 'https://drs.microbiomedata.org/objects/', 'nmpdr': 'http://www.nmpdr.org/FIG/wiki/rest.cgi/NmpdrPlugin/SeedViewer?page=Annotation;feature=', 'NMR': 'http://purl.obolibrary.org/obo/NMR_', 'nmrcv': 'http://purl.obolibrary.org/obo/NMR_', 'nmrshiftdb2': 'https://nmrshiftdb.nmr.uni-koeln.de/molecule/', 'noaa': 'https://www.fisheries.noaa.gov/species/', 'NOMEN': 'http://purl.obolibrary.org/obo/NOMEN_', 'noncodev3': 'http://www.noncode.org/NONCODERv3/ncrna.php?ncid=', 'noncodev4.gene': 'http://www.bioinfo.org/NONCODEv4/show_gene.php?id=', 'noncodev4.rna': 'http://www.bioinfo.org/NONCODEv4/show_rna.php?id=', 'norine': 'http://bioinfo.lifl.fr/norine/result.jsp?ID=', 'novus': 'https://www.novusbio.com/products/', 'npo': 'http://purl.bioontology.org/ontology/npo#NPO_', 'nsc': 'https://dtp.cancer.gov/dtpstandard/servlet/ChemData?searchtype=NSC&searchlist=', 'nsf.award': 'https://www.nsf.gov/awardsearch/showAward?AWD_ID=', 'nucleardb': 'http://www.receptors.org/nucleardb/proteins/', 'nuclearbd': 'http://www.receptors.org/nucleardb/proteins/', 'oa': 'http://www.w3.org/ns/oa#', 'OAE': 'http://purl.obolibrary.org/obo/OAE_', 'OARCS': 'http://purl.obolibrary.org/obo/OARCS_', 'OBA': 'http://purl.obolibrary.org/obo/OBA_', 'oban': 'http://purl.org/oban/', 'OBCS': 'http://purl.obolibrary.org/obo/OBCS_', 'OBI': 'http://purl.obolibrary.org/obo/OBI_', 'OBIB': 'http://purl.obolibrary.org/obo/OBIB_', 'oboInOwl': 'http://www.geneontology.org/formats/oboInOwl#', 'oboformat': 'http://www.geneontology.org/formats/oboInOwl#', 'oio': 'http://www.geneontology.org/formats/oboInOwl#', 'obv': 'https://permalink.obvsg.at/', 'occ': 'https://w3id.org/oc/corpus/', 'oci': 'https://w3id.org/oc/oci/', 'ocid': 'https://ocid.ontochem.com/prefname?ocid=', 'oclc': 'https://www.worldcat.org/oclc/', 'odam': 'https://identifiers.org/odam:', 'odc.sci': 'https://odc-sci.org/data/', 'odc.tbi': 'https://odc-tbi.org/data/', 'odor': 'https://identifiers.org/odor:', 'odrl': 'http://www.w3.org/ns/odrl/2/', 'OGG': 'http://purl.obolibrary.org/obo/OGG_', 'OGI': 'http://purl.obolibrary.org/obo/OGI_', 'OGI.owl': 'http://purl.obolibrary.org/obo/OGI_', 'OGMS': 'http://purl.obolibrary.org/obo/OGMS_', 'ogms/OMRE': 'http://purl.obolibrary.org/obo/OGMS_', 'OGSF': 'http://purl.obolibrary.org/obo/OGSF_', 'OHD': 'http://purl.obolibrary.org/obo/OHD_', 'OHMI': 'http://purl.obolibrary.org/obo/OHMI_', 'OHPI': 'http://purl.obolibrary.org/obo/OHPI_', 'oid': 'http://oid-info.com/get/', 'OlatDv': 'http://purl.obolibrary.org/obo/OlatDv_', 'om': 'http://www.ontology-of-units-of-measure.org/resource/om-2/', 'oma.grp': 'https://omabrowser.org/cgi-bin/gateway.pl?f=DisplayGroup&p1=', 'oma.hog': 'https://omabrowser.org/oma/hog/HOG:', 'oma.protein': 'https://omabrowser.org/cgi-bin/gateway.pl?f=DisplayEntry&p1=', 'omia': 'https://omia.org/OMIA', 'OMIABIS': 'http://purl.obolibrary.org/obo/OMIABIS_', 'omid': 'https://w3id.org/oc/meta/', 'omim': 'https://omim.org/MIM:', 'OMIM': 'https://omim.org/MIM:', 'mim': 'https://omim.org/MIM:', 'omim.ps': 'https://omim.org/MIM:PS', 'MIMPS': 'https://omim.org/MIM:PS', 'OMIMPS': 'https://omim.org/MIM:PS', 'PS': 'https://omim.org/MIM:PS', 'mim.ps': 'https://omim.org/MIM:PS', 'OMIT': 'http://purl.obolibrary.org/obo/OMIT_', 'OMO': 'http://purl.obolibrary.org/obo/OMO_', 'omop': 'https://athena.ohdsi.org/search-terms/terms/', 'OMOP': 'https://athena.ohdsi.org/search-terms/terms/', 'OMP': 'http://purl.obolibrary.org/obo/OMP_', 'OMRSE': 'http://purl.obolibrary.org/obo/OMRSE_', 'oncotree': 'http://oncotree.mskcc.org/api/tumorTypes/search/code/', 'ONE': 'http://purl.obolibrary.org/obo/ONE_', 'ONS': 'http://purl.obolibrary.org/obo/ONS_', 'ontie': 'https://ontology.iedb.org/ontology/ONTIE_', 'ONTOAVIDA': 'http://purl.obolibrary.org/obo/ONTOAVIDA_', 'ONTONEO': 'http://purl.obolibrary.org/obo/ONTONEO_', 'OOSTT': 'http://purl.obolibrary.org/obo/OOSTT_', 'opb': 'https://bioportal.bioontology.org/ontologies/OPB/?p=classes&conceptid=http%3A%2F%2Fbhi.washington.edu%2FOPB%23OPB_', 'openalex': 'https://openalex.org/', 'OPL': 'http://purl.obolibrary.org/obo/OPL_', 'opm': 'http://opm.phar.umich.edu/protein.php?pdbid=', 'OPMI': 'http://purl.obolibrary.org/obo/OPMI_', 'orcid': 'https://orcid.org/', 'ORCID': 'https://orcid.org/', 'ORCiD': 'https://orcid.org/', 'ordb': 'http://senselab.med.yale.edu/ORDB/Data/', 'oridb.sacch': 'http://cerevisiae.oridb.org/details.php?id=', 'oridb.schizo': 'http://pombe.oridb.org/details.php?id=', 'ORNASEQ': 'http://purl.obolibrary.org/obo/ORNASEQ_', 'orphanet': 'http://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=', 'ORPHA': 'http://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=', 'orphanet.ordo': 'http://www.orpha.net/ORDO/Orphanet_', 'ordo': 'http://www.orpha.net/ORDO/Orphanet_', 'orth': 'http://purl.org/net/orth#', 'oryzabase.gene': 'http://www.shigen.nig.ac.jp/rice/oryzabaseV4/gene/detail/', 'oryzabase.mutant': 'http://www.shigen.nig.ac.jp/rice/oryzabaseV4/strain/inducedMutationLine/detail/', 'oryzabase.reference': 'https://shigen.nig.ac.jp/rice/oryzabase/reference/detail/', 'oryzabase.stage': 'http://www.shigen.nig.ac.jp/rice/oryzabaseV4/devstageineachorgan/detail/', 'oryzabase.strain': 'http://www.shigen.nig.ac.jp/rice/oryzabaseV4/strain/wildCore/detail/', 'oslc': 'http://open-services.net/ns/core#', 'otl': 'http://oryzatagline.cirad.fr/cgi-bin/general_mutant.pl?line=', 'otol': 'https://tree.opentreeoflife.org/taxonomy/browse?id=', 'OVAE': 'http://purl.obolibrary.org/obo/OVAE_', 'owl': 'http://www.w3.org/2002/07/owl#', 'owlstar': 'http://w3id.org/owlstar/', 'p3db.protein': 'https://identifiers.org/p3db.protein:', 'p3db.site': 'https://identifiers.org/p3db.site:', 'paleodb': 'http://fossilworks.org/?a=taxonInfo&taxon_no=', 'panther.family': 'http://www.pantherdb.org/panther/family.do?clsAccession=', 'panther.node': 'http://www.pantree.org/node/annotationNode.jsp?id=', 'panther.pathway': 'http://www.pantherdb.org/pathway/pathwayDiagram.jsp?catAccession=', 'panther.pthcmp': 'http://www.pantherdb.org/pathway/pathCatDetail.do?clsAccession=', 'PAO': 'http://purl.obolibrary.org/obo/PAO_', 'pass2': 'http://caps.ncbs.res.in/cgi-bin/pass2//show_sf.py?sf_id=', 'pathbank': 'https://pathbank.org/view/', 'pathguide': 'http://www.pathguide.org/fullrecord.php?organisms=all&availability=all&standards=all&order=alphabetic&DBID=', 'pathoplant': 'http://www.pathoplant.de/detail.php?accNo=', 'pathwaycommons': 'http://www.pathwaycommons.org/pc/record2.do?id=', 'PATO': 'http://purl.obolibrary.org/obo/PATO_', 'pav': 'http://purl.org/pav/', 'paxdb.organism': 'http://pax-db.org/#!species/', 'paxdb.protein': 'http://pax-db.org/#!protein/', 'pazar': 'http://www.pazar.info/cgi-bin/tf_search.cgi?geneID=', 'PBA': 'https://biopragmatics.github.io/providers/pba/', 'PCL': 'http://purl.obolibrary.org/obo/PCL_', 'PCO': 'http://purl.obolibrary.org/obo/PCO_', 'PD_ST': 'http://purl.obolibrary.org/obo/PD_ST_', 'pdb': 'https://rdf.wwpdb.org/pdb/', 'RCSB_PDB': 'https://rdf.wwpdb.org/pdb/', 'pdbe': 'https://rdf.wwpdb.org/pdb/', 'pdbj': 'https://rdf.wwpdb.org/pdb/', 'pdbsum': 'https://rdf.wwpdb.org/pdb/', 'validatordb': 'https://rdf.wwpdb.org/pdb/', 'viperdb': 'https://rdf.wwpdb.org/pdb/', 'wwpdb': 'https://rdf.wwpdb.org/pdb/', 'pdb.ligand': 'http://www.rcsb.org/pdb/ligand/ligandsummary.do?hetId=', 'PDB_CHEM_ID': 'http://www.rcsb.org/pdb/ligand/ligandsummary.do?hetId=', 'pdb-ccd': 'http://www.rcsb.org/pdb/ligand/ligandsummary.do?hetId=', 'pdbechem': 'http://www.rcsb.org/pdb/ligand/ligandsummary.do?hetId=', 'pdc.study': 'https://pdc.cancer.gov/pdc/study/', 'PDRO': 'http://purl.obolibrary.org/obo/PDRO_', 'PdumDv': 'http://purl.obolibrary.org/obo/PdumDv_', 'PECO': 'http://purl.obolibrary.org/obo/PECO_', 'ped': 'https://proteinensemble.org/', 'ped.ensemble': 'https://proteinensemble.org/', 'peff': 'https://biopragmatics.github.io/providers/peff/', 'pennsieve': 'https://discover.pennsieve.io/package/', 'pepbank': 'http://pepbank.mgh.harvard.edu/interactions/details/', 'peptideatlas': 'https://db.systemsbiology.net/sbeams/cgi/PeptideAtlas/Summarize_Peptide?query=QUERY&searchForThis=', 'peptideatlas.dataset': 'http://www.peptideatlas.org/PASS/', 'perkinelmer': 'https://www.perkinelmer.com/searchresult?searchName=', 'peroxibase': 'http://peroxibase.toulouse.inra.fr/browse/process/view_perox.php?id=', 'pfam': 'https://www.ebi.ac.uk/interpro/entry/pfam/', 'PF': 'https://www.ebi.ac.uk/interpro/entry/pfam/', 'pandit': 'https://www.ebi.ac.uk/interpro/entry/pfam/', 'pfam.clan': 'https://www.ebi.ac.uk/interpro/set/pfam/', 'pfr': 'http://repository.topdownproteomics.org/proteoforms/', 'TDR': 'http://repository.topdownproteomics.org/proteoforms/', 'PGDSO': 'http://purl.obolibrary.org/obo/PGDSO_', 'pgs': 'https://www.pgscatalog.org/pgs/', 'pgx': 'https://progenetix.org/services/ids/', 'Progenetix': 'https://progenetix.org/services/ids/', 'pharmacodb.cell': 'https://pharmacodb.ca/cell_lines/', 'pharmacodb.dataset': 'https://pharmacodb.ca/datasets/', 'pharmacodb.tissue': 'https://pharmacodb.ca/tissues/', 'pharmgkb.disease': 'http://www.pharmgkb.org/disease/', 'pharmgkb.drug': 'http://www.pharmgkb.org/drug/', 'pharmgkb.gene': 'http://www.pharmgkb.org/gene/', 'pharmgkb.pathways': 'http://www.pharmgkb.org/pathway/', 'phenolexplorer': 'http://phenol-explorer.eu/foods/', 'phenx': 'https://www.phenxtoolkit.org/protocols/view/', 'phenxtoolkit': 'https://www.phenxtoolkit.org/protocols/view/', 'PHIPO': 'http://purl.obolibrary.org/obo/PHIPO_', 'phosphopoint.kinase': 'https://identifiers.org/phosphopoint.kinase:', 'phosphopoint.protein': 'https://identifiers.org/phosphopoint.protein:', 'phosphosite.protein': 'http://www.phosphosite.org/proteinAction.do?id=', 'phosphosite.residue': 'http://www.phosphosite.org/siteAction.do?id=', 'phylomedb': 'http://phylomedb.org/?seqid=', 'phytozome.locus': 'http://www.phytozome.net/genePage.php?crown&method=0&search=1&detail=1&searchText=locusname:', 'JGI Phytozome': 'http://www.phytozome.net/genePage.php?crown&method=0&search=1&detail=1&searchText=locusname:', 'pibase': 'http://modbase.compbio.ucsf.edu/pibase-cgi/get_details.pl?object_type=complexes&bdp_id=', 'pid.pathway': 'http://pid.nci.nih.gov/search/pathway_landing.shtml?what=graphic&jpg=on&pathway_id=', 'pid': 'http://pid.nci.nih.gov/search/pathway_landing.shtml?what=graphic&jpg=on&pathway_id=', 'piroplasma': 'http://piroplasmadb.org/piro/showRecord.do?name=GeneRecordClasses.GeneRecordClass&source_id=', 'pirsf': 'https://pir.georgetown.edu/cgi-bin/ipcSF?id=', 'pkdb': 'https://pk-db.com/data/', 'PLANA': 'http://purl.obolibrary.org/obo/PLANA_', 'PLANP': 'http://purl.obolibrary.org/obo/PLANP_', 'planttfdb': 'http://planttfdb.cbi.pku.edu.cn/tf.php?uid=', 'plasmodb': 'http://plasmodb.org/plasmo/showRecord.do?name=GeneRecordClasses.GeneRecordClass&source_id=', 'ApiDB_PlasmoDB': 'http://plasmodb.org/plasmo/showRecord.do?name=GeneRecordClasses.GeneRecordClass&source_id=', 'PLO': 'http://purl.obolibrary.org/obo/PLO_', 'pmap.cutdb': 'http://cutdb.burnham.org/relation/show/', 'pmap.substratedb': 'https://identifiers.org/pmap.substratedb:', 'pmc': 'http://europepmc.org/articles/', 'PMC': 'http://europepmc.org/articles/', 'PMCID': 'http://europepmc.org/articles/', 'pmdb': 'https://bioinformatics.cineca.it/PMDB/user//search.php?idsearch=', 'pmr': 'https://models.physiomeproject.org/exposure/', 'pmr.workspace': 'https://models.physiomeproject.org/workspace/', 'PO': 'http://purl.obolibrary.org/obo/PO_', 'pocketome': 'https://identifiers.org/pocketome:', 'polbase': 'https://identifiers.org/polbase:', 'pombase': 'https://www.pombase.org/gene/', 'PomBase': 'https://www.pombase.org/gene/', 'PORO': 'http://purl.obolibrary.org/obo/PORO_', 'PPO': 'http://purl.obolibrary.org/obo/PPO_', 'ppr': 'https://europepmc.org/article/ppr/', 'PR': 'http://purl.obolibrary.org/obo/PR_', 'PRO': 'http://purl.obolibrary.org/obo/PR_', 'prefixcommons': 'https://bioregistry.io/metaregistry/prefixcommons/', 'pride': 'http://purl.obolibrary.org/obo/PRIDE_', 'pride.project': 'https://www.ebi.ac.uk/pride/archive/projects/', 'prints': 'https://identifiers.org/prints:', 'sprint': 'https://identifiers.org/prints:', 'probesanddrugs': 'https://www.probes-drugs.org/compound/', 'probonto': 'http://www.probonto.org/ontology#PROB_', 'PROCO': 'http://purl.obolibrary.org/obo/PROCO_', 'prodom': 'http://prodom.prabi.fr/prodom/current/cgi-bin/request.pl?question=DBEN&query=', 'proglyc': 'http://www.proglycprot.org/detail.aspx?ProId=', 'PROPREO': 'http://purl.obolibrary.org/obo/PROPREO_', 'prosite': 'https://prosite.expasy.org/', 'protclustdb': 'https://www.ncbi.nlm.nih.gov/sites/entrez?Db=proteinclusters&Cmd=DetailsSearch&Term=', 'protcom': 'http://www.ces.clemson.edu/compbio/protcom/print_file.cgi?pdbid=', 'proteomicsdb.peptide': 'https://www.proteomicsdb.org/#human/proteinDetails/', 'proteomicsdb.protein': 'https://identifiers.org/proteomicsdb.protein:', 'protonet.cluster': 'http://www.protonet.cs.huji.ac.il/requested/cluster_card.php?cluster=', 'protonet.proteincard': 'http://www.protonet.cs.huji.ac.il/requested/protein_card.php?protein_id=', 'prov': 'http://www.w3.org/ns/prov#', 'pscdb': 'https://identifiers.org/pscdb:', 'PSDO': 'http://purl.obolibrary.org/obo/PSDO_', 'pseudogene': 'http://tables.pseudogene.org/[?species_name]/', 'pseudogene.org': 'http://tables.pseudogene.org/[?species_name]/', 'pseudomonas': 'http://www.pseudomonas.com/feature/show/?locus_tag=', 'psipar': 'https://www.ebi.ac.uk/ontology-lookup/?termId=PAR:', 'PSO': 'http://purl.obolibrary.org/obo/PSO_', 'pubchem.bioassay': 'https://pubchem.ncbi.nlm.nih.gov/bioassay/', 'pubchem.aid': 'https://pubchem.ncbi.nlm.nih.gov/bioassay/', 'pubchem.assay': 'https://pubchem.ncbi.nlm.nih.gov/bioassay/', 'pubchem.cell': 'https://pubchem.ncbi.nlm.nih.gov/cell/', 'pubchem.classification': 'https://pubchem.ncbi.nlm.nih.gov/classification/#hid=', 'pubchem.compound': 'http://rdf.ncbi.nlm.nih.gov/pubchem/compound/CID', 'CID': 'http://rdf.ncbi.nlm.nih.gov/pubchem/compound/CID', 'DSSTox_CID': 'http://rdf.ncbi.nlm.nih.gov/pubchem/compound/CID', 'PUBCHEM_CID': 'http://rdf.ncbi.nlm.nih.gov/pubchem/compound/CID', 'PubChem_Compound_CID': 'http://rdf.ncbi.nlm.nih.gov/pubchem/compound/CID', 'Pubchem': 'http://rdf.ncbi.nlm.nih.gov/pubchem/compound/CID', 'pubchem_id': 'http://rdf.ncbi.nlm.nih.gov/pubchem/compound/CID', 'pubchem.element': 'https://pubchem.ncbi.nlm.nih.gov/element/', 'pubchem.substance': 'http://rdf.ncbi.nlm.nih.gov/pubchem/substance/SID', 'DSSTox_Generic_SID': 'http://rdf.ncbi.nlm.nih.gov/pubchem/substance/SID', 'publons.publication': 'https://publons.com/publon/', 'publons.researcher': 'https://publons.com/researcher/', 'PMID': 'http://rdf.ncbi.nlm.nih.gov/pubchem/reference/', 'MEDLINE': 'http://rdf.ncbi.nlm.nih.gov/pubchem/reference/', 'PubMed': 'http://rdf.ncbi.nlm.nih.gov/pubchem/reference/', 'pmid': 'http://rdf.ncbi.nlm.nih.gov/pubchem/reference/', 'PuRO': 'http://purl.org/spar/pro/', 'PW': 'http://purl.obolibrary.org/obo/PW_', 'PWO': 'http://purl.org/spar/pwo/', 'px': 'http://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=', 'pypi': 'https://pypi.org/project/', 'qb': 'http://purl.org/linked-data/cube#', 'qtldb': 'https://www.animalgenome.org/QTLdb/q?id=QTL_ID:', 'cattleqtldb': 'https://www.animalgenome.org/QTLdb/q?id=QTL_ID:', 'chickenqtldb': 'https://www.animalgenome.org/QTLdb/q?id=QTL_ID:', 'pigqtldb': 'https://www.animalgenome.org/QTLdb/q?id=QTL_ID:', 'sheepqtldb': 'https://www.animalgenome.org/QTLdb/q?id=QTL_ID:', 'qudt': 'http://qudt.org/schema/qudt#', 'radiomics': 'http://www.radiomics.org/RO/', 'radlex': 'https://radlex.org/RID/', 'rapdb.locus': 'https://rapdb.dna.affrc.go.jp/viewer/gbrowse_details/irgsp1?name=', 'irgsp': 'https://rapdb.dna.affrc.go.jp/viewer/gbrowse_details/irgsp1?name=', 'rapdb.transcript': 'https://rapdb.dna.affrc.go.jp/viewer/gene_detail/irgsp1?name=', 'ratmap': 'http://ratmap.org/Showgene.php?gene_stable_id=', 'rbk': 'https://www.rebuildingakidney.org/id/', 'RBO': 'http://purl.obolibrary.org/obo/RBO_', 'rcb': 'https://cellbank.brc.riken.jp/cell_bank/CellInfo/?cellNo=', 'rdf': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#', 'rdfs': 'http://www.w3.org/2000/01/rdf-schema#', 'rdo': 'http://www.semanticweb.org/mca/ontologies/2018/8/untitled-ontology-47#', 're3data': 'https://www.re3data.org/repository/', 'reactome': 'https://reactome.org/content/detail/', 'RE': 'https://reactome.org/content/detail/', 'REACT': 'https://reactome.org/content/detail/', 'rebase': 'https://identifiers.org/rebase:', 'rebec': 'https://ensaiosclinicos.gov.br/rg/', 'receptome.family': 'http://www.receptome.org/families/Frameset_family.asp?ProtType=Receptor&FamId=', 'redfly': 'http://redfly.ccr.buffalo.edu/?content=/view_detail.php&crm_id=', 'repeatsdb.structure': 'https://repeatsdb.org/structure/', 'repec': 'https://www.ins.gob.pe/ensayosclinicos/rpec/recuperarECPBNuevo.asp?val=&NroPag=1&flg=0&ver=EN&numEC=', 'reproduceme': 'https://w3id.org/reproduceme#', 'RESID': 'http://purl.obolibrary.org/obo/RESID_', 'REX': 'http://purl.obolibrary.org/obo/REX_', 'rfam': 'https://rfam.xfam.org/family/', 'rfc': 'https://tools.ietf.org/rfc/rfc', 'rgap': 'http://rice.plantbiology.msu.edu/cgi-bin/gbrowse/rice/?name=', 'rgd': 'http://rgd.mcw.edu/rgdweb/report/gene/main.html?id=', 'RGD': 'http://rgd.mcw.edu/rgdweb/report/gene/main.html?id=', 'rgd.qtl': 'http://rgd.mcw.edu/rgdweb/report/qtl/main.html?id=', 'rgd.strain': 'http://rgd.mcw.edu/rgdweb/report/strain/main.html?id=', 'rhea': 'https://www.rhea-db.org/rhea/', 'RHEA': 'https://www.rhea-db.org/rhea/', 'ribocentre': 'https://www.ribocentre.org/docs/', 'ribocenter': 'https://www.ribocentre.org/docs/', 'ricecyc': 'http://pathway.gramene.org/RICE/NEW-IMAGE?type=PATHWAY&object=', 'ricegap': 'http://rice.plantbiology.msu.edu/cgi-bin/ORF_infopage.cgi?&orf=', 'ricenetdb.compound': 'http://bis.zju.edu.cn/ricenetdb/compounddetails.php?ID=', 'ricenetdb.gene': 'http://bis.zju.edu.cn/ricenetdb/genedetails.php?ID=', 'ricenetdb.mirna': 'http://bis.zju.edu.cn/ricenetdb/miRNAdetails.php?ID=', 'ricenetdb.protein': 'http://bis.zju.edu.cn/ricenetdb/proteindetails.php?ID=', 'ricenetdb.reaction': 'http://bis.zju.edu.cn/ricenetdb/reactiondetails.php?ID=', 'rism': 'https://rism.online/', 'rna_sstrand': 'http://www.rnasoft.ca/sstrand/show_results.php?molecule_ID=', 'rnacentral': 'https://rnacentral.org/rna/', 'LNCRNADB': 'https://rnacentral.org/rna/', 'rnajunction': 'https://rnajunction.ncifcrf.gov/JunctionPage.php?jid=', 'rnaloops': 'https://rnaloops.cs.put.poznan.pl/search/details/', 'rnamod': 'http://rna.rega.kuleuven.be/cgi-bin/rnamods/rnashow.pl?', 'rnamods': 'http://mods.rna.albany.edu/mods/modifications/view/', 'RNAO': 'http://purl.obolibrary.org/obo/RNAO_', 'rnavdb': 'http://virus.zoo.ox.ac.uk/rnavirusdb/virus.php?id=', 'RO': 'http://purl.obolibrary.org/obo/RO_', 'RO_proposed_relation': 'http://purl.obolibrary.org/obo/RO_', 'obo_rel': 'http://purl.obolibrary.org/obo/RO_', 'oborel': 'http://purl.obolibrary.org/obo/RO_', 'RoleO': 'http://purl.obolibrary.org/obo/RoleO_', 'ror': 'https://ror.org/', 'rouge': 'https://www.kazusa.or.jp/rouge/gfpage/', 'rpcec': 'https://rpcec.sld.cu/en/trials/', 'rrid': 'https://scicrunch.org/resolver/RRID:', 'RS': 'http://purl.obolibrary.org/obo/RS_', 'runbiosimulations': 'https://run.biosimulations.org/simulations/', 'RXNO': 'http://purl.obolibrary.org/obo/RXNO_', 'rxnorm': 'https://mor.nlm.nih.gov/RxNav/search?searchBy=RXCUI&searchTerm=', 'RXCUI': 'https://mor.nlm.nih.gov/RxNav/search?searchBy=RXCUI&searchTerm=', 's_mart_db': 'http://smartdb.bioinf.med.uni-goettingen.de/cgi-bin/SMARtDB/getSMAR.cgi?', 'sabiork.compound': 'http://sabiork.h-its.org/newSearch?q=', 'sabiork.kineticrecord': 'http://sabiork.h-its.org/kineticLawEntry.jsp?viewData=true&kinlawid=', 'sabiork.reaction': 'http://sabiork.h-its.org/reacdetails.jsp?reactid=', 'SABIO-RK': 'http://sabiork.h-its.org/reacdetails.jsp?reactid=', 'salk': 'https://abrc.osu.edu/stocks/number/SALK_', 'salmon': 'https://purl.dataone.org/odo/SALMON_', 'SAO': 'http://purl.obolibrary.org/obo/SAO_', 'sasap': 'https://purl.dataone.org/odo/SASAP_', 'sasbdb': 'http://www.sasbdb.org/data/', 'SBO': 'http://purl.obolibrary.org/obo/SBO_', 'SCDO': 'http://purl.obolibrary.org/obo/SCDO_', 'schem': 'https://biopragmatics.github.io/providers/schem/', 'schema': 'https://schema.org/', 'schemaorg': 'https://schema.org/', 'scholia.resource': 'https://bioregistry.io/metaregistry/scholia/', 'sciflection': 'https://sciflection.com/', 'scomp': 'https://biopragmatics.github.io/providers/scomp/', 'scop': 'http://scop.berkeley.edu/sunid=', 'scop.sun': 'http://scop.berkeley.edu/sunid=', 'scop.sunid': 'http://scop.berkeley.edu/sunid=', 'scop.sccs': 'http://scop.berkeley.edu/sccs=', 'scop.sid': 'http://scop.berkeley.edu/sid=', 'scopus': 'https://www.scopus.com/authid/detail.uri?authorId=', 'scopus.author': 'https://www.scopus.com/authid/detail.uri?authorId=', 'scopus.affiliation': 'https://www.scopus.com/affil/profile.uri?afid=', 'scopus.publication': 'https://www.scopus.com/sourceid/', 'scopus.work': 'http://www.scopus.com/record/display.url?origin=inward&eid=', 'scopus.eid': 'http://www.scopus.com/record/display.url?origin=inward&eid=', 'SCoRO': 'http://purl.org/spar/scoro/', 'scpd': 'http://rulai.cshl.edu/cgi-bin/SCPD/getgene2?', 'scr': 'http://scicrunch.org/resolver/SCR_', 'scicrunch': 'http://scicrunch.org/resolver/SCR_', 'scretf': 'http://stormo.wustl.edu/ScerTF/details/', 'sdap': 'http://fermi.utmb.edu/cgi-bin/SDAP/sdap_02?dB_Type=0&allid=', 'sdbs': 'http://riodb01.ibase.aist.go.jp/sdbs/cgi-bin/cre_frame_disp.cgi?sdbsno=', 'sdgio': 'http://purl.unep.org/sdg/SDGIO_', 'sdis': 'https://biopragmatics.github.io/providers/sdis/', 'sedml.format': 'https://sed-ml.org/urns.html#format:', 'sedml.language': 'https://sed-ml.org/urns.html#language:', 'seed': 'http://seed-viewer.theseed.org/seedviewer.cgi?page=Subsystems&subsystem=', 'seed.compound': 'http://modelseed.org/biochem/compounds/', 'seed.reaction': 'http://modelseed.org/biochem/reactions/', 'seinet': 'https://swbiodiversity.org/seinet/taxa/index.php?taxon=', 'semapv': 'https://w3id.org/semapv/vocab/', 'senso': 'http://purl.dataone.org/odo/SENSO_', 'seo': 'https://w3id.org/seo#', 'SEP': 'http://purl.obolibrary.org/obo/SEP_', 'SEPIO': 'http://purl.obolibrary.org/obo/SEPIO_', 'sfam': 'https://biopragmatics.github.io/providers/sfam/', 'sgd': 'https://www.yeastgenome.org/locus/', 'sgd.pathways': 'http://pathway.yeastgenome.org/YEAST/new-image?type=PATHWAY&object=', 'sgn': 'http://solgenomics.net/phenome/locus_display.pl?locus_id=', 'sh': 'http://www.w3.org/ns/shacl#', 'shacl': 'http://www.w3.org/ns/shacl#', 'shareloc': 'https://shareloc.xyz/#/ontology/vocabulary/SHARELOC_', 'sharkipedia.species': 'https://www.sharkipedia.org/species/', 'sharkipedia.trait': 'https://www.sharkipedia.org/traits/', 'sharkipedia.trend': 'https://www.sharkipedia.org/trends/', 'shex': 'http://www.w3.org/ns/shex#', 'shibase': 'http://www.mgc.ac.cn/cgi-bin/ShiBASE/ShiBASE_query.cgi?synonym=', 'SIBO': 'http://purl.obolibrary.org/obo/SIBO_', 'sider.drug': 'http://sideeffects.embl.de/drugs/', 'sigmaaldrich': 'https://www.sigmaaldrich.com/US/en/product/sigma/', 'signaling-gateway': 'http://www.signaling-gateway.org/molecule/query?afcsid=', 'signor': 'https://signor.uniroma2.it/relation_result.php?id=', 'SIGNOR': 'https://signor.uniroma2.it/relation_result.php?id=', 'sio': 'https://www.ebi.ac.uk/ols4/ontologies/sio/terms?short_form=', 'sisu': 'http://search.sisuproject.fi/#/variant/', 'sitex': 'http://www-bionet.sscc.ru/sitex/index.php?siteid=', 'skip': 'https://skip.stemcellinformatics.org/SKIPSearch/cell_line_detail?accession=', 'skm': 'https://identifiers.org/skm:', 'skos': 'http://www.w3.org/2004/02/skos/core#', 'skosxl': 'http://www.w3.org/2008/05/skos-xl#', 'slkbase': 'https://sumlineknowledgebase.com/?page_id=', 'SLM': 'https://www.swisslipids.org/#/entity/SLM:', 'swisslipid': 'https://www.swisslipids.org/#/entity/SLM:', 'swisslipids': 'https://www.swisslipids.org/#/entity/SLM:', 'smart': 'http://smart.embl-heidelberg.de/smart/do_annotation.pl?DOMAIN=', 'smid': 'https://smid-db.org/smid/', 'SMID-DB': 'https://smid-db.org/smid/', 'smiles': 'https://chemapps.stolaf.edu/jmol/jmol.php?model=', 'smpdb': 'https://smpdb.ca/view/', 'SMP': 'https://smpdb.ca/view/', 'snap': 'http://www.ifomis.org/bfo/1.1/snap#', 'SCTID': 'http://snomed.info/id/', 'SCTID_2010_1_31': 'http://snomed.info/id/', 'SNOMECT': 'http://snomed.info/id/', 'SNOMED': 'http://snomed.info/id/', 'SNOMEDCT': 'http://snomed.info/id/', 'SNOMEDCTCT': 'http://snomed.info/id/', 'SNOMEDCTCT_2018_03_01': 'http://snomed.info/id/', 'SNOMEDCTCT_2019_03_01': 'http://snomed.info/id/', 'SNOMEDCT_2005_07_31': 'http://snomed.info/id/', 'SNOMEDCT_2010_1_31': 'http://snomed.info/id/', 'SNOMEDCT_2020_03_01': 'http://snomed.info/id/', 'SNOMEDCT_US': 'http://snomed.info/id/', 'SNOMEDCT_US_2015_03_01': 'http://snomed.info/id/', 'SNOMEDCT_US_2016_03_01': 'http://snomed.info/id/', 'SNOMEDCT_US_2018_03_01': 'http://snomed.info/id/', 'SNOMEDCT_US_2018_09_01': 'http://snomed.info/id/', 'SNOMEDCT_US_2019_03_01': 'http://snomed.info/id/', 'SNOMEDCT_US_2019_09_01': 'http://snomed.info/id/', 'SNOMEDCT_US_2020_03_01': 'http://snomed.info/id/', 'SNOMEDCT_US_2020_09_01': 'http://snomed.info/id/', 'SNOMEDCT_US_2021_03_01': 'http://snomed.info/id/', 'SNOMEDCT_US_2021_07_31': 'http://snomed.info/id/', 'SNOMEDCT_US_2021_09_01': 'http://snomed.info/id/', 'SNOMEDCT_US_2022_07_31': 'http://snomed.info/id/', 'SNOMEDCT_US_2022_09_01': 'http://snomed.info/id/', 'SNOMEDCT_US_2022_12_31': 'http://snomed.info/id/', 'SNOMEDCT_US_2023_02_28': 'http://snomed.info/id/', 'SNOMED_CT': 'http://snomed.info/id/', 'SNOMED_CT_US_2018_03_01': 'http://snomed.info/id/', 'SNOWMEDCT': 'http://snomed.info/id/', 'SNOWMEDCT_US': 'http://snomed.info/id/', 'SNOWMEDCT_US_2018_03_01': 'http://snomed.info/id/', 'snornabase': 'http://www-snorna.biotoul.fr/plus.php?id=', 'snp500cancer': 'http://snp500cancer.nci.nih.gov/snp.cfm?both_snp_id=', 'SO': 'http://purl.obolibrary.org/obo/SO_', 'SOPHARM': 'http://purl.obolibrary.org/obo/SOPHARM_', 'soybase': 'http://www.soybase.org/sbt/search/search_results.php?category=SNP&search_term=', 'spbase': 'http://www.spbase.org/SpBase/search/viewAnnoGeneInfo.php?spu_id=', 'SPD': 'http://purl.obolibrary.org/obo/SPD_', 'spdx': 'https://spdx.org/licenses/', 'sphn': 'https://biomedit.ch/rdf/sphn-ontology/sphn#', 'spike.map': 'https://identifiers.org/spike.map:', 'splash': 'http://mona.fiehnlab.ucdavis.edu/#/spectra/splash/', 'splicenest': 'http://splicenest.molgen.mpg.de/cgi-bin/splicegraph.cgi?cluster=', 'spp': 'https://www.signalingpathways.org/datasets/dataset.jsf?doi=', 'srao': 'http://www.fairsharing.org/ontology/subject/SRAO_', 'ssbd.dataset': 'https://ssbd.riken.jp/database/dataset/', 'ssbd.project': 'https://ssbd.riken.jp/database/project/', 'sssom': 'https://w3id.org/sssom/', 'sstoss': 'http://caps.ncbs.res.in/cgi-bin/mini/databases/SSTOSS/sstoss.cgi?code=', 'stap': 'http://psb.kobic.re.kr/STAP/refinement1/result.php?search=', 'STATO': 'http://purl.obolibrary.org/obo/STATO_', 'stitch': 'http://stitch.embl.de/interactions/', 'storedb': 'https://www.storedb.org/?', 'storedb.dataset': 'https://www.storedb.org/?STOREDB:DATASET', 'storedb.file': 'https://www.storedb.org/store_v3/download.jsp?fileId=', 'storedb.study': 'https://www.storedb.org/store_v3/study.jsp?studyId=', 'sty': 'https://uts.nlm.nih.gov/uts/umls/semantic-network/', 'UMLSSG': 'https://uts.nlm.nih.gov/uts/umls/semantic-network/', 'umls.st': 'https://uts.nlm.nih.gov/uts/umls/semantic-network/', 'umls.sty': 'https://uts.nlm.nih.gov/uts/umls/semantic-network/', 'subtilist': 'http://genolist.pasteur.fr/SubtiList/genome.cgi?external_query+', 'subtiwiki': 'http://www.subtiwiki.uni-goettingen.de/wiki/index.php/', 'sugarbind': 'http://sugarbind.expasy.org/', 'supfam': 'http://supfam.org/SUPERFAMILY/cgi-bin/scop.cgi?ipid=', 'sweetrealm': 'http://sweetontology.net/matrMineral/', 'swh': 'https://archive.softwareheritage.org/browse/swh:', 'swissregulon': 'http://swissregulon.unibas.ch/query/', 'SWO': 'http://purl.obolibrary.org/obo/SWO_', 'swrl': 'http://www.w3.org/2003/11/swrl#', 'SYMP': 'http://purl.obolibrary.org/obo/SYMP_', 't3db': 'http://www.t3db.org/toxins/', 'T4FS': 'http://purl.obolibrary.org/obo/T4FS_', 'TADS': 'http://purl.obolibrary.org/obo/TADS_', 'TAHE': 'http://purl.obolibrary.org/obo/TAHE_', 'TAHH': 'http://purl.obolibrary.org/obo/TAHH_', 'tair.gene': 'http://arabidopsis.org/servlets/TairObject?accession=Gene:', 'tair.locus': 'http://www.arabidopsis.org/servlets/TairObject?accession=Locus:', 'tair.protein': 'http://arabidopsis.org/servlets/TairObject?accession=AASequence:', 'TAO': 'http://purl.obolibrary.org/obo/TAO_', 'TAO_RETIRED': 'http://purl.obolibrary.org/obo/TAO_', 'tarbase': 'http://diana.imis.athena-innovation.gr/DianaTools/index.php?r=tarbase/index&mirnas=', 'TAXRANK': 'http://purl.obolibrary.org/obo/TAXRANK_', 'tcb': 'https://www.pirbright.ac.uk/node/', 'tccd': 'http://www.itb.cnr.it/cellcycle/gene_rep.php?gene_name=', 'tcdb': 'http://www.tcdb.org/search/result.php?tc=', 'TC': 'http://www.tcdb.org/search/result.php?tc=', 'tctr': 'https://www.thaiclinicaltrials.org/show/', 'tfclass': 'http://tfclass.bioinf.med.uni-goettingen.de/?tfclass=', 'tgd': 'http://ciliate.org/index.php/feature/details/', 'TGMA': 'http://purl.obolibrary.org/obo/TGMA_', 'tgn': 'http://vocab.getty.edu/page/tgn/', 'thermofisher': 'https://www.thermofisher.com/antibody/product/', 'tigrfam': 'https://www.ncbi.nlm.nih.gov/cdd?term=', 'time': 'http://www.w3.org/2006/time#', 'tngb': 'http://biobanknetwork.telethon.it/Sample/View?sampleId=', 'TO': 'http://purl.obolibrary.org/obo/TO_', 'togovar': 'https://grch38.togovar.org/variant/', 'tol': 'http://tolweb.org/', 'tol.webproj': 'http://tolweb.org/', 'topdb': 'http://topdb.enzim.hu/?m=show&id=', 'toxoplasma': 'http://toxodb.org/toxo/showRecord.do?name=GeneRecordClasses.GeneRecordClass&source_id=', 'TRANS': 'http://purl.obolibrary.org/obo/TRANS_', 'transportdb': 'http://www.membranetransport.org/protein.php?pOID=mmar1&pSynonym=', 'transyt': 'https://transyt.bio.di.uminho.pt/reactions/', 'tred': 'http://rulai.cshl.edu/cgi-bin/TRED/tred.cgi?process=geneInfo&gid=', 'treebase': 'https://identifiers.org/treebase:', 'treefam': 'http://www.treefam.org/family/', 'trichdb': 'http://trichdb.org/trichdb/showRecord.do?name=GeneRecordClasses.GeneRecordClass&source_id=', 'tritrypdb': 'http://tritrypdb.org/tritrypdb/showRecord.do?name=GeneRecordClasses.GeneRecordClass&source_id=', 'trnadbce': 'http://trna.nagahama-i-bio.ac.jp/cgi-bin/trnadb/whole_detail.cgi?SID=', 'ttd.drug': 'http://bidd.nus.edu.sg/group/TTD/ZFTTDDRUG.asp?ID=', 'ttd.target': 'http://bidd.nus.edu.sg/group/TTD/ZFTTDDetail.asp?ID=', 'TTO': 'http://purl.obolibrary.org/obo/TTO_', 'TXPO': 'http://purl.obolibrary.org/obo/TXPO_', 'UBERON': 'http://purl.obolibrary.org/obo/UBERON_', 'Uber-anatomy ontology': 'http://purl.obolibrary.org/obo/UBERON_', 'Uberon': 'http://purl.obolibrary.org/obo/UBERON_', 'ubio.namebank': 'http://www.ubio.org/browser/details.php?namebankID=', 'ucsc': 'ftp://hgdownload.cse.ucsc.edu/goldenPath/', 'ucum': 'https://units-of-measurement.org/', 'umbbd.compound': 'http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=c&compID=', 'UM-BBD_compID': 'http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=c&compID=', 'umbbd.enzyme': 'http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=ep&enzymeID=', 'UM-BBD_enzymeID': 'http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=ep&enzymeID=', 'umbbd.pathway': 'http://umbbd.ethz.ch/servlets/pageservlet?ptype=p&pathway_abbr=', 'UM-BBD_pathwayID': 'http://umbbd.ethz.ch/servlets/pageservlet?ptype=p&pathway_abbr=', 'umbbd.reaction': 'http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=r&reacID=', 'UM-BBD_reactionID': 'http://eawag-bbd.ethz.ch/servlets/pageservlet?ptype=r&reacID=', 'umbbd.rule': 'http://www.umbbd.ethz.ch/servlets/rule.jsp?rule=', 'UM-BBD_ruleID': 'http://www.umbbd.ethz.ch/servlets/rule.jsp?rule=', 'UMLS': 'http://linkedlifedata.com/resource/umls/id/', 'UMLS CUI': 'http://linkedlifedata.com/resource/umls/id/', 'UMLS_CUI': 'http://linkedlifedata.com/resource/umls/id/', 'sider.effect': 'http://linkedlifedata.com/resource/umls/id/', 'umls.cui': 'http://linkedlifedata.com/resource/umls/id/', 'unichem': 'https://www.ebi.ac.uk/unichem/compoundsources?type=uci&compound=', 'unigene': 'http://www.ncbi.nlm.nih.gov/UniGene/clust.cgi?UGID=', 'unii': 'https://precision.fda.gov/uniisearch/srs/unii/', 'unimod': 'http://www.unimod.org/modifications_view.php?editid1=', 'uniparc': 'https://www.ebi.ac.uk/cgi-bin/dbfetch?db=uniparc&id=', 'unipathway.compound': 'http://www.grenoble.prabi.fr/obiwarehouse/unipathway/upc?upid=', 'unipathway.reaction': 'http://www.grenoble.prabi.fr/obiwarehouse/unipathway/ucr?upid=', 'uniprot': 'http://purl.uniprot.org/uniprot/', 'SwissProt': 'http://purl.uniprot.org/uniprot/', 'UP': 'http://purl.uniprot.org/uniprot/', 'UniProt': 'http://purl.uniprot.org/uniprot/', 'UniProtKB': 'http://purl.uniprot.org/uniprot/', 'Uniprot ID': 'http://purl.uniprot.org/uniprot/', 'apid.interactions': 'http://purl.uniprot.org/uniprot/', 'bykdb': 'http://purl.uniprot.org/uniprot/', 'compulyeast': 'http://purl.uniprot.org/uniprot/', 'goa': 'http://purl.uniprot.org/uniprot/', 'mobidb': 'http://purl.uniprot.org/uniprot/', 'orthodb': 'http://purl.uniprot.org/uniprot/', 'pina': 'http://purl.uniprot.org/uniprot/', 'pmp': 'http://purl.uniprot.org/uniprot/', 'repeatsdb.protein': 'http://purl.uniprot.org/uniprot/', 'string': 'http://purl.uniprot.org/uniprot/', 'swiss-model': 'http://purl.uniprot.org/uniprot/', 'topfind': 'http://purl.uniprot.org/uniprot/', 'uniprot/swiss-prot': 'http://purl.uniprot.org/uniprot/', 'uniprot.arba': 'https://www.uniprot.org/arba/', 'arba': 'https://www.uniprot.org/arba/', 'uniprot.chain': 'http://purl.uniprot.org/annotation/', 'UPPRO': 'http://purl.uniprot.org/annotation/', 'uniprot.disease': 'https://www.uniprot.org/diseases/', 'DI': 'https://www.uniprot.org/diseases/', 'SP_DI': 'https://www.uniprot.org/diseases/', 'uniprot.isoform': 'http://purl.uniprot.org/isoforms/', 'UPISO': 'http://purl.uniprot.org/isoforms/', 'uniprot.journal': 'http://purl.uniprot.org/journals/', 'uniprot.keyword': 'https://www.uniprot.org/keywords/', 'SP_KW': 'https://www.uniprot.org/keywords/', 'UniProtKB-KW': 'https://www.uniprot.org/keywords/', 'uniprot.kw': 'https://www.uniprot.org/keywords/', 'uniprot.location': 'https://www.uniprot.org/locations/', 'SP_SL': 'https://www.uniprot.org/locations/', 'UPLOC': 'https://www.uniprot.org/locations/', 'UniProtKB-SubCell': 'https://www.uniprot.org/locations/', 'uniprot.proteome': 'https://www.uniprot.org/proteomes/', 'uniprot.ptm': 'https://biopragmatics.github.io/providers/uniprot.ptm/', 'PTM': 'https://biopragmatics.github.io/providers/uniprot.ptm/', 'uniprot.resource': 'https://www.uniprot.org/database/', 'uniprot.database': 'https://www.uniprot.org/database/', 'uniprot.db': 'https://www.uniprot.org/database/', 'uniprot.tissue': 'https://www.uniprot.org/tissues/', 'uniprot.var': 'http://purl.uniprot.org/annotation/VAR_', 'SP_VAR': 'http://purl.uniprot.org/annotation/VAR_', 'UPVAR': 'http://purl.uniprot.org/annotation/VAR_', 'UniProtKB_VAR': 'http://purl.uniprot.org/annotation/VAR_', 'uniref': 'https://www.uniprot.org/uniref/', 'unirule': 'https://www.uniprot.org/unirule/', 'unists': 'https://www.ncbi.nlm.nih.gov/genome/sts/sts.cgi?uid=', 'unite': 'http://unite.ut.ee/bl_forw.php?nimi=', 'UO': 'http://purl.obolibrary.org/obo/UO_', 'UPA': 'http://purl.obolibrary.org/obo/UPA_', 'UPa': 'http://purl.obolibrary.org/obo/UPA_', 'unipathway': 'http://purl.obolibrary.org/obo/UPA_', 'unipathway.pathway': 'http://purl.obolibrary.org/obo/UPA_', 'UPHENO': 'http://purl.obolibrary.org/obo/UPHENO_', 'uspto': 'http://patft.uspto.gov/netacgi/nph-Parser?Sect2=PTO1&Sect2=HITOFF&p=1&u=/netahtml/PTO/search-bool.html&r=1&f=G&l=50&d=PALL&RefSrch=yes&Query=PN/', 'utrdb': 'https://utrdb.cloud.ba.infn.it/cgi-bin/utrdb/utrdb?e=', 'vac': 'https://vac.niaid.nih.gov/view?id=', 'vandf': 'http://purl.bioontology.org/ontology/VANDF/', 'vann': 'https://vocab.org/vann/', 'VariO': 'http://purl.obolibrary.org/obo/VariO_', 'vbase2': 'http://www.vbase2.org/vgene.php?id=', 'VBO': 'http://purl.obolibrary.org/obo/VBO_', 'vbrc': 'http://vbrc.org/gene_detail.asp?gene_id=', 'vcell': 'https://vcell.org/biomodel-', 'vectorbase': 'https://vectorbase.org/gene/', 'vega': 'http://vega.sanger.ac.uk/[?species_name]/Gene/Summary?g=', 'VEGA': 'http://vega.sanger.ac.uk/[?species_name]/Gene/Summary?g=', 'vegbank': 'http://vegbank.org/cite/', 'vfb': 'http://virtualflybrain.org/reports/', 'vfdb.gene': 'http://www.mgc.ac.cn/cgi-bin/VFs/gene.cgi?GeneID=', 'vfdb.genus': 'http://www.mgc.ac.cn/cgi-bin/VFs/genus.cgi?Genus=', 'vgnc': 'https://vertebrate.genenames.org/data/gene-symbol-report/#!/vgnc_id/', 'VHOG': 'http://purl.obolibrary.org/obo/VHOG_', 'VHOG_RETIRED': 'http://purl.obolibrary.org/obo/VHOG_', 'viaf': 'http://viaf.org/viaf/', 'vido': 'http://purl.obolibrary.org/obo/VIDO_', 'violinnet': 'http://www.violinet.org/vaxquery/vaccine_detail.php?c_vaccine_id=', 'violinID': 'http://www.violinet.org/vaxquery/vaccine_detail.php?c_vaccine_id=', 'violinId': 'http://www.violinet.org/vaxquery/vaccine_detail.php?c_vaccine_id=', 'vipr': 'https://identifiers.org/vipr:', 'viralzone': 'https://viralzone.expasy.org/', 'vz': 'https://viralzone.expasy.org/', 'viroligo': 'http://viroligo.okstate.edu/main.php?vid=', 'virsirna': 'http://crdd.osdd.net/servers/virsirnadb/record.php?details=', 'vita': 'http://vita.mbc.nctu.edu.tw/search.php?acc=', 'vmhgene': 'https://www.vmh.life/#gene/', 'vmhmetabolite': 'https://www.vmh.life/#metabolite/', 'vmhreaction': 'https://www.vmh.life/#reaction/', 'VO': 'http://purl.obolibrary.org/obo/VO_', 'void': 'http://rdfs.org/ns/void#', 'VSAO': 'http://purl.obolibrary.org/obo/VSAO_', 'VSAO_RETIRED': 'http://purl.obolibrary.org/obo/VSAO_', 'vsmo': 'http://purl.obolibrary.org/obo/VSMO_', 'vso': 'https://bioportal.bioontology.org/ontologies/VSO/?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FVSO_', 'VT': 'http://purl.obolibrary.org/obo/VT_', 'VTO': 'http://purl.obolibrary.org/obo/VTO_', 'wb.rnai': 'https://www.wormbase.org/species/c_elegans/rnai/', 'WBbt': 'http://purl.obolibrary.org/obo/WBbt_', 'WBls': 'http://purl.obolibrary.org/obo/WBls_', 'WBPhenotype': 'http://purl.obolibrary.org/obo/WBPhenotype_', 'webelements': 'https://www.webelements.com/', 'wgs84': 'http://www.w3.org/2003/01/geo/wgs84_pos#', 'wikidata': 'http://www.wikidata.org/entity/', 'WD_Entity': 'http://www.wikidata.org/entity/', 'wd': 'http://www.wikidata.org/entity/', 'wikidata.property': 'https://www.wikidata.org/wiki/Property:', 'wikipathways': 'http://identifiers.org/wikipathways/', 'wikipathways.vocab': 'https://bioportal.bioontology.org/ontologies/WIKIPATHWAYS/?p=classes&conceptid=http%3A%2F%2Fvocabularies.wikipathways.org%2Fwp%23', 'wikipedia.en': 'http://en.wikipedia.org/wiki/', 'wikipedia': 'http://en.wikipedia.org/wiki/', 'worfdb': 'http://worfdb.dfci.harvard.edu/index.php?search_type=name&page=showresultrc&race_query=', 'world2dpage': 'https://world-2dpage.expasy.org/repository/', 'worldavatar.kin': 'http://www.theworldavatar.com/ontology/ontokin/OntoKin.owl#', 'WormBase': 'https://www.wormbase.org/get?name=', 'WB_REF': 'https://www.wormbase.org/get?name=', 'wb': 'https://www.wormbase.org/get?name=', 'wormbase': 'https://www.wormbase.org/get?name=', 'wormpep': 'https://www.wormbase.org/db/seq/protein?name=', 'worms': 'http://www.marinespecies.org/aphia.php?p=taxdetails&id=', 'wwf.ecoregion': 'https://www.worldwildlife.org/ecoregions/', 'WWF': 'https://www.worldwildlife.org/ecoregions/', 'XAO': 'http://purl.obolibrary.org/obo/XAO_', 'XCO': 'http://purl.obolibrary.org/obo/XCO_', 'xenbase': 'https://www.xenbase.org/entry/', 'Xenbase': 'https://www.xenbase.org/entry/', 'ximbio': 'https://ximbio.com/reagent/', 'XLMOD': 'http://purl.obolibrary.org/obo/XLMOD_', 'xmetdb': 'http://www.xmetdb.org/xmetdb/protocol/', 'xml': 'http://www.w3.org/XML/1998/namespace#', 'XPO': 'http://purl.obolibrary.org/obo/XPO_', 'xsd': 'http://www.w3.org/2001/XMLSchema#', 'ydpm': 'http://www-deletion.stanford.edu/cgi-bin/YDPM/YDPM_search.cgi?thelist=', 'yeastintron': 'http://intron.ucsc.edu/cgi-bin/yeast4.3/intronreports.pl?outputformat=full&amp;orfName=', 'yeastract': 'http://www.yeastract.com/view.php?existing=locus&orfname=', 'yetfasco': 'http://yetfasco.ccbr.utoronto.ca/showPFM.php?mot=', 'yid': 'http://metarray.ucsc.edu/cgi-bin/intron/yirIntrondb?orfName=', 'ymdb': 'http://www.ymdb.ca/compounds/', 'YPO': 'http://purl.obolibrary.org/obo/YPO_', 'yrcpdr': 'http://yeastrc.org/pdr/viewProtein.do?id=', 'ZEA': 'http://purl.obolibrary.org/obo/ZEA_', 'ZECO': 'http://purl.obolibrary.org/obo/ZECO_', 'zenodo.record': 'https://zenodo.org/record/', 'zenodo': 'https://zenodo.org/record/', 'ZFA': 'http://purl.obolibrary.org/obo/ZFA_', 'ZFA_RETIRED': 'http://purl.obolibrary.org/obo/ZFA_', 'zfin': 'http://zfin.org/', 'ZFIN': 'http://zfin.org/', 'ZFS': 'http://purl.obolibrary.org/obo/ZFS_', 'zinc': 'http://zinc15.docking.org/substances/', 'ZP': 'http://purl.obolibrary.org/obo/ZP_'}

```

Reply:
Now even the internal[ sssom-schema context](https://github.com/mapping-commons/sssom/blob/79b7f3ebc4939ddd32a4f53cc626f0e8427e0127/src/sssom_schema/context/sssom_schema.context.jsonld#L3-L15) seems to exist in the `bioregistry` context. Does that mean this iteration we just make one source of truth i.e. `bioregistry`?

Reply:
All exaxt prefix uri prefix pairs in the sssom schema context are already included in the bioregistry context, even if you don't merge the contxt variable?

Reply:
Ok I think this is basically the same as what was there before then when using the bimap. Lets merge this, and do the bigger refactoring relying entirely on the epm in a subsequent PR.

Reply:
I think using `uri_prefix` should basically replicate exactly what was there before.

Reply:
Maybe sanity check some cases.